---
layout: article
title: Of Prions and People
date: '2005-10-16 23:15:19 -0700'
author: Andrea Bottaro
mt_id: 1443
blog_id: 2
post_id: 1443
basename: of_prions_and_p
---
Spurred by a host of new findings in molecular and cellular biology, in recent years an increasing number of determined biologists have come to envision processes that contradict century-old biological assumptions and seem to defy the expectations of Darwinian evolutionary theory...

_Naaah_, I am not talking about ID.  I am talking about prions, the specter of Jean Baptiste de Lamarck, and "heretical" views about biology.    And what must be truly baffling for conspiracy-minded ID advocates, the inflexible "Darwinist orthodoxy" seems to positively _dig_ this "heresy".  Now, _that_ must hurt...

For many years after their discovery as the agents of some rare neurodegenerative diseases in mammals, such as _scrapie_ in sheep and human Creutzfeldt-Jakob disease and _kuru_, prions have remained a truly esoteric research topic. However, they have recently become well-known to the public as the cause of bovine spongiform encephalopathy, a.k.a. BSE or "mad cow disease", which can be occasionally transmitted to humans causing variant Creutzfeld-Jacob disease (vCJD). (Information about prion diseases can be found at the web sites of the [WHO](http://www.who.int/zoonoses/diseases/prion_diseases/en/) and the [National Prion Disease Pathology Surveillance Center](http://www.cjdsurveillance.com/).) 

Prions are unlike any other infectious agent in that they seem to have no nucleic acids at all.  Indeed, after a long controversy, most scientists currently agree that prions propagate entirely as alternatively folded forms of certain proteins, through a mechanism that resembles crystal nucleation (see figure below) \[4,8,12\].

<figure>
<a href="http://www.pandasthumb.org/archives/prions%20JPEG.JPG"><img src="http://www.pandasthumb.org/archives/images/prionssm.jpg" alt="prionssm.jpg" width="414" height="149" /></a>
<figcaption markdown="span">
Prion formation and replication.  Click on image for larger version.

</figcaption>
</figure>

In the neurological diseases mentioned above, a prion protein can cause a "normal" folded protein of the same type (called PrPc for _prion protein - cellular form_) to assume a prion-like conformation. (For a PT primer on protein folding and some recent findings, see [here](http://www.pandasthumb.org/archives/2005/10/protein_folding.html).) Misfolded prion-like proteins tend to form filaments, and cause conformation changes in additional PrPc's they come in contact with.  This results in an amplifying cascade that eventually leads to accumulation of prion protein fibers ("amyloid") which cause progressive neuronal cell death, and ultimately disease manifestation.  Transfer of prion proteins _alone_ from a sick organism to a healthy one causes the propagation cascade to start anew.  

Let's pause here.  Does something nag you?  It should.  When prions propagate, what gets _replicated_ is not really a material entity.  Non-pathogenetic, correctly folded PrPc proteins exist in a host _before_ prion infection.  Unlike all other infectious agents, prions do not make new forms of themselves by _synthesizing_ anything.  Like in the case of crystal formation, what gets replicated is a _structure_.  In other words, prions are not replicating proteins, but replicating _shapes_ in a protein substrate world.  There is even evidence that the same prion may exist in alternative forms ("strains") with different properties, which may potentially compete with each other for the available substrates (PrPc's), in a sort of Darwinian competition between "immaterial" replicators.   Now, this is not unexpected: Darwinain evolution is a logically unavoidable consequence of replicators displaying heritable variation in a selective environment.  Nevertheless, it's just spooky, if you ask me.  But that's not all.

People have found prions not only in mammals, but also in other organisms, such as yeast and other fungi.  Fungal prions are a great system to study prion biology, because they can be manipulated in the lab without excessive concern with potential infectivity, and their host organisms replicate and express the prion phenotype very fast (unlike the lag time for prion disease symptom development in mammals).  In addition, fungal prions have attracted considerable interest because they actually mediate heritable phenotypes in their natural hosts, phenotypes that in some cases can be adaptive. 

For instance, a yeast prion called \[PSI+\] causes defects in protein synthesis termination, and formation of new proteins \[10, 11, 6, 8\].  As most of you know, ribosomes are the cellular organelles which assemble proteins by linking amino acids according to the nucleotide sequence of messenger RNAs.  In a nutshell, \[PSI+\] prevents the ribosome from reading the RNA nucleotide triplets (_codons_) that encode for "stop" signals.  This causes elongation of the encoded proteins past their normal stop sites (i.e., proteins acquire extra amino acid sequences at their tail end), or in some cases allows translation of pseudogene transcripts that are otherwise crippled by stop codons.  

In normal conditions, this phenotype spontaneously appears at low frequency in yeast populations, and is unstable (\[PSI+\] cells can revert to normal at low rates).  The \[PSI+\] phenotype is generally non-adaptive (in fact, mal-adaptive), but in certain conditions (for instance, if the environment changes drastically) some of the aberrant proteins it generates may confer advantageous new functions.  If that happens, prion-bearing organisms capable of synthesizing the new advantageous proteins will spread through the population.  These phenotypically altered cells, by surviving in the new conditions, have now a chance to acquire favorable _genetic_ mutations through conventional mutation processes.  If this happens, the gene mutation can take over the population, while the \[PSI+\] population is again counter-selected and essentially disappears.  

The \[PSI+\] prion trait is transmitted in a non-mendelian fashion.  When mating \[PSI+\] and \[psi-\] yeast strains, the progeny is \[PSI+\], and so are, counter to Mendel's laws, all the progeny of this progeny (except for the low-frequency "reversion" rate to the normal phenotype).   The possibility of generating the \[PSI+\] trait is genetically encoded in the sequence of the yeast gene for the normal version of the \[PSI+\] prion protein, called Sup35 (which has the ability to assume both prion and non-prion folds). Therefore, although the prion "option" is clearly subject to conventional Darwinian evolution, in the case of \[PSI+\] _natural selection is acting on a non-mendelian, non-genetically encoded trait_. 

One of the most interesting aspects of these prion-dependent phenomena is that they seem to have evolved because they provide a "buffer" system against sudden environmental changes by allowing the rapid generation of large numbers of new phenotypes, one of which may turn out to be useful in the new selective conditions. Another mechanism with similar properties is the SOS response in bacteria, in which hypermutation occurs in environmentally stressful situations.   "Evolvability" is an often misued term that refers to the evolutionary potential of certain traits or organisms, but it applies well in this case, to indicate the increased diversity of otherwise "hidden" phenotypes that are unmasked by the \[PSI+\] state \[4-8\].

Several scientists have also noted that prion-dependent phenotypes raise the very real possibility of specific direct environmental induction of a heritable phenotypic trait, a kind of _Lamarckian evolution_, or pre-programmed phenotypic switch \[2-4,6,8,12\].  In certain conditions, for instance when an organisms recurrently but unpredictably encounters a specific, strong selective condition, prion systems may result in the environmental induction of adaptive, acquired heritable phenotypes. 

At this point, we don't have any _bona fide_ examples of this actually happening, but in principle it's possible.  Moreover, this model provides a real, testable mechanism to explain such a phenomenon, should it occur (scientists don't mind testable mechanisms and hypotheses, [even when they are "heretical"](http://www.pandasthumb.org/archives/2004/04/getting_away_with_heresy.html)).  
<img src="http://www.pandasthumb.org/archives/Aplysia_JPEG.JPG" alt="Aplysia_JPEG.JPG" width="420" height="336" style="float:left;" />
In fact, something that comes tantalizingly close to this, at the cellular, if not organismic level, has been proposed to occur during long-term memory formation in the sea slug _Aplysia californica_\[1,8,9\].  This critter has been used experimentally for many years as a model for memory formation for its rudimentary learning processes and giant neurons.  Long-term memory formation has been associated with the formation of stable functional contacts (synapses) between neurons, in a form of competition: many synapses form all the time, but only those that get progressively stabilized by repeated stimulation will persist.  One of the proteins that has been associated with the synapse stabilization process in Aplysia is called ApCPEB.  Its function is somewhat unclear but, like Sup35, it also may be involved in control of protein translation.  ApCPEB localizes to the synapse region, and upon repeated synaptic stimulation it forms "clusters" with different regulatory properties compared to the monomeric protein.   Once formed, these clusters remain stable despite protein turn-over, even in the absence of further stimulatory signals.  In yeast, ApCPEB has been shown to act as a _bona fide_ prion, which would explain the formation of clusters and their stable properties.

These findings have raised enormous interest, not only from prion specialists, but among biologists in general.  Many papers have appeared in major journals discussing the data and their implications.  The extent of these phenomena in the biological world is unclear: prion-based inheritance and evolution may be extremely rare, or perhaps it's quite pervasive and we just missed it.  Some scientists are already talking about "paradigm shifts" \[e.g., 2\], although that's probably premature.   Regardless, it certainly runs against the impression, which ID proponents are trying to project in their P.R. _communiques_, of a monolithic, censorial Darwinian orthodoxy bent on stifling dissent and hiding evidence inconsistent with mainstream evolutionary theory.  It is hard to say at this point whether in 50 years evolutionary biology textbooks will devote prions a whole chapter, a page, or just a footnote.  But prions, unlike ID, will most like be there.  

**References**
1. Bailey CH, Kandel ER, Si K. The persistence of long-term memory: a molecular approach to self-sustaining changes in learning-induced synaptic growth. Neuron. 2004 44:49-57. 

2. Bussard AE. A scientific revolution? The prion anomaly may challenge the central dogma of molecular biology. EMBO Rep. 2005 6:691-4. 

3. Chernoff YO. Mutation processes at the protein level: is Lamarck back? Mutat Res. 2001 488:39-64. 

4. Chernoff YO. Replication vehicles of protein-based inheritance. Trends Biotechnol. 2004 22:549-52. 

5. Chicurel M. Genetics. Can organisms speed their own evolution? Science. 2001 292:1824-7. 

6. Masel J, Bergman A. The evolution of the evolvability properties of the yeast prion \[PSI+\]. Evolution Int J Org Evolution. 2003 57:1498-512.  

7. Partridge L, Barton NH. Evolving evolvability. Nature. 2000 407:457-8. 

8. Shorter J, Lindquist S. Prions as adaptive conduits of memory and inheritance. Nat Rev Genet. 2005 6:435-50. 

9. Si K, Lindquist S, Kandel ER. A neuronal isoform of the aplysia CPEB has prion-like properties. Cell. 2003 115:879-91. 

10. True HL, Lindquist SL. A yeast prion provides a mechanism for genetic variation and phenotypic diversity. Nature. 2000 407:477-83. 

11. True HL, Berlin I, Lindquist SL. Epigenetic regulation of translation reveals hidden genetic variation to produce complex traits. Nature. 2004 431:184-7. 

12. Uptain SM, Lindquist S. Prions as protein-based genetic elements. Annu Rev Microbiol. 2002 56:703-41.
