---
layout: article
title: 'Human Evolution: It''s all in the testes'
date: '2005-12-14 20:20:08 -0700'
excerpt: <a ref="http://biology.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pbio.0030399">Retrocopied
  Genes May Enhance Male Fitness</a>
author: Matt Brauer
mt_id: 1675
blog_id: 2
post_id: 1675
basename: human_evolution
---
[<img src="{{ site.baseurl }}/uploads/2006/plosMarques-thumb.jpg" alt="plosMarques.jpg" width="135" height="173" style="float:right;" />](/uploads/2006/plosMarques.jpg)
Review of: [
Marques et al., "Emergence of Young Human Genes after a Burst of Retroposition in Primates."](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030357-S.pdf)
**PLoS Biology 3**(11):1970-1979.([Synopsis ](http://biology.plosjournals.org/perlserv/?request=get-document&amp;doi=10.1371/journal.pbio.0030399) on PLoS Biology)

[
November's issue of PLoS biology](http://biology.plosjournals.org/perlserv/?request=get-toc&amp;issn=1545-7885&amp;volume=3&amp;issue=11) has several papers of evolutionary relevance. 



* Richard Robinson gives a [
nice review](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030396-S.pdf)
of some current thinking about [
abiogenesis](http://wiki.cotch.net/index.php/Abiogenesis).
* The
[
evolution of "genetic robustness"](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030381-S.pdf)
is explored in a paper from Paul Turner's lab.
* A paper by 
[
Sabeti ET al.](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030378-S.pdf)
demonstrates that the evolution of a disease resistance locus in humans, thought to have been under strong recent selection, cannot actually be distinguished as non-neutral.
* Mating preferences in fruit flies were shown by 
[
Rundle ET al.](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030368-S.pdf)
to evolve as a side effect of selection in divergent environments.


In addition, [this very elegant paper](http://biology.plosjournals.org/perlserv/?request=get-pdf&amp;file=10.1371_journal.pbio.0030357-S.pdf) describes some surprising results relating to the evolution of new genes in humans.

Considered at even moderate timescales the genome is a dynamic structure. Chromosomes get rearranged, duplicated or lost, and genes pop in and out of existence like virtual particles. One of the mechanisms for the movement of genes is the so-called retrotransposable element, or [_retrotransposon_](http://en.wikipedia.org/wiki/Retrotransposon). Along with their cousins the [transposons](http://en.wikipedia.org/wiki/Transposon), retrotransposons are parasites on the genome, and rely upon the cellular machinery to copy their sequences to new locations. (In some cases, transposable elements retain a structure that is suggestive of an origin from a viral genome. Just as tapeworms have lost the superfluous parts required for life outside the host, transposons appear to be viruses that have shed their corporeal existence in order to live as a string of bits in someone else's genome.) Retrotransposons look a lot like degenerate retrovirus genomes. They also replicate similarly, passing first through an mRNA cycle (transcribed by the host's polymerase) and then re-integrating as DNA  synthesized with a retrotransposon-provided Reverse Transcriptase (RT).

The presence of the enzyme RT has an unusual side-effect: occasionally a molecule of the cell's mRNA will act as a substrate, and the message will be converted back to DNA. There is an important difference between the original gene and its reverse-transcribed child, however: while the original gene might have had introns that interrupted the sequence, the retrotranscribed child molecule derives from a processed mRNA, and so is intron-free. Because of this, the parent gene can be distinguished from the child. (In addition, the child gene most likely will have integrated at a new place in the genome where it would not have been under useful cellular control, and it would have degenerated into a pseudogene as a result.)

One family of retrotransposons, known as LINE elements, appears to have gone through a phase of high activity in several mammalian lineages about 100 Myears ago. The activity was so high that up to 20% of some mammalian sequences derives from LINE element, in fact. This burst of activity presumably resulted in a lot of RT appearing in the nuclei of various mammalian cells at that time. So much so, in fact, that the incidence of "retroposition" -- this copying of a gene to another location via mRNA -- seems to have increased dramatically, most notably in the primates, including humans.

What became of these retrocopies? Like most copies, they degenerated into shadows of their former selves. But in some few cases, they retained activity. These retrocopies presumably landed in a portion of the genome where the the transcriptional regulation had a positive effect on the organism. This is expected to happen rarely, and indeed the event appears to be uncommon. Marques et al. found that only about 18% of identified retrocopies appeared to be intact. (Many more retrocopies, made invisible by extensive degradation, may have been missed, of course.) Based on the rate of neutral mutation, they estimated that most of these retrocopies originated at around the same time, about 40-50 MYA. This is after the primate-mouse split, and is consistent with a burst of retroposition occurring in primates at about that time.

The authors went on to examine the fate of intact (and presumably still functional) retrocopies. They found that the 38 genes they examined arose at various points during primate evolution. Seven of these genes were unlikely to have remained intact in all lineages without the action of some selection. These seven were therefore identified as candidate functional retrogenes.

What further can be said about these candidates? The authors looked at the tissues where the parents of these genes were expressed, and compared them to the location of the functional retrogenes.

They found that the parents of the seven genes were expressed in all tissues. But in **every single case**, the corresponding retrogene was **expressed in testes only**.

It's fun to imagine the authors' sense of glee as this realization dawned on them. For there is no way that they ought to have expected this to be the case. The genes were chosen by criteria that were independent of the location of expression. To find that **all** of the genes are expressed in the same tissue, and only in that tissue, is a phenomenal result.

So what does it mean? There is some idea that retrogene formation may be favored in testes due to hyperactivity of transcription during male meiosis. But these are genes that have been under functional selection, so unless all retrogenes _originated_ in testes, the odds of seeing this particular group of seven there are very low.

For five of these genes, the answer may be related to the fact that the parental genes lie on the X chromosome. Half of the male gametes lack the X chromosome, so the retrocopies might have originally been making up for this intrinsic lack. Both of the other retrogenes have parents involved in spermatogenesis, so it is reasonable to assume that the extra copies add functionality to the testes that carry them.

Many of the postulated mechanisms of rapid evolutionary change (sperm competition, reinforcement, varieties of sexual selection) relate to male reproductive function. Thus it is gratifying to see that the phenomenal source of genetic variation supplied by retrogenes is most apparent in the testes.

The next chapter in this story should be the identification of the new functions that these genes may have adopted in the testes. By doing this, we'll have another very strong example of the purposeful arrangement of parts effected by a known, defined and purposeless mechanism.
