---
layout: article
title: Casey Luskin's "Junk" Arguments
date: '2008-09-28 11:56:39 -0700'
author: PvM
mt_id: 3971
blog_id: 2
post_id: 3971
basename: casey_luskins_j
---
In a classic 'bait and switch', Casey Luskin, 'argues' that the "classic "Junk DNA" icon of neo-Darwinism needs updating" because a Yale University news release shows how differences in the regulatory elements between humans and chimps explain the human thumb and foot development.

In a classic misdirect, Luskin, a lawyer by training, includes regulatory elements as "junk DNA" even though the original definition of "Junk DNA" only included pseudo-genes, and even though regulatory elements were since long known not to be 'junk DNA', in fact the conservation of said regulatory elements was something that rendered it anything but 'junk'. 

Casey's "junk" argument continues to conflate and confuse as to what neo-Darwinian theory says about it, and includes the customary quote-mining while avoiding actual understanding.

An example

Casey Luskin wrote:

> In 2006, Michael Shermer asserted, "Rather than being intelligently designed, the human genome looks more and more like a mosaic of mutations, fragment copies, borrowed sequences, and discarded strings of DNA that were jerry-built over millions of years of evolution." \[2\]

\[2.\] Michael Shermer, Why Darwin Matters: The Case Against Intelligent Design, pg. 75 (Times Books 2006). 

Now what is the actual argument by Shermer?

> DNA is information, and if the Law of Conservation of Information requires the input of an Intelligent Designer in order to increase specified complexity of the genome, we have to wonder why the Intelligent Designer added to our genome junk DNA, repeated copies of useless DNA, orphan genes, tandem repeats, and pseudogenes, none of which are involved directly in the making of a human being.


Why does Luskin fail to inform us of the context of the argument? Well, the answer is really simple: The facts do not support his vacuous claims about neo-Darwinism and "Junk DNA". To argue that people did not know that there are countless regulatory elements 'hidden' in the "Junk DNA" ignores the history.

For those interested in the real science behind these findings, please read John Timmer's 
[Genomic study finds enhancer of human hands (or feet)](http://arstechnica.com/journals/science.ars/2008/09/11/genomic-study-finds-enhancer-of-human-hands-or-feet)  or [Did a gene enhancer humanise our thumbs?](http://scienceblogs.com/notrocketscience/2008/09/did_a_gene_enhancer_humanise_our_thumbs.php) at Not Rocket Science.

The Yale [news release](http://opa.yale.edu/news/article.aspx?id=5980) may help explain Luskin's "confusion" but he should know that rather than accepting news releases, one should look at the actual papers to arrive at one's conclusions.

In the journal science Science, Wray and Babbitt provide their [perspectives](http://www.sciencemag.org/cgi/content/summary/321/5894/1300) on the study.

> **Abstract**: Nearly half a century has passed since François Jacob and Jacques Monod demonstrated that specific noncoding sequences are required to activate genes that metabolize lactose in the bacterium Escherichia coli (1). In a prescient observation, they noted that mutations in these regulatory sequences might play a role in the evolution of organismal traits. They further argued that gene function is not only based on the biochemical activity of its product but also on how the gene's expression is regulated. This idea was expanded in 1975 in an influential paper by Mary-Claire King and Alan Wilson (2), who proposed that trait differences between humans and chimpanzees are primarily due to regulatory changes in gene expression. Decades elapsed, however, before it was feasible to begin testing these ideas in detail. Two papers in this issue, by Prabhakar et al. on page 1346 (3), and by Hong et al. on page 1314 (4), demonstrate the power of combining bioinformatic approaches with experimental tests to characterize such regulatory regions.
> 
> A major impediment to studying the evolutionary importance of mutations in regulatory regions is simply knowing where to look. DNA sequences that regulate the transcription of genes occupy no fixed position relative to coding DNA regions and are often diffuse and widely dispersed. Even when the position of a regulatory element is known, there is the added challenge of identifying which mutations have functional consequences. Within coding sequences, the genetic code imposes familiar regularities: Mutations that change protein structure can be identified exhaustively and unambiguously. By contrast, identifying functional mutations within regulatory regions requires experimental tests of putative regulatory elements from different species or individuals--a costly and time-consuming process. Bioinformatic methods offer a way to identify promising functional noncoding regions and to narrow the focus for experimental tests.

The study in question  was also published in science as [Human-Specific Gain of Function in a Developmental Enhancer](http://www.sciencemag.org/cgi/content/abstract/sci;321/5894/1346) Science 5 September 2008: Vol. 321. no. 5894, pp. 1346 - 1350

> **Abstract:** §
> 
> Changes in gene regulation are thought to have contributed to the evolution of human development. However, in vivo evidence for uniquely human developmental regulatory function has remained elusive. In transgenic mice, a conserved noncoding sequence (HACNS1) that evolved extremely rapidly in humans acted as an enhancer of gene expression that has gained a strong limb expression domain relative to the orthologous elements from chimpanzee and rhesus macaque. This gain of function was consistent across two developmental stages in the mouse and included the presumptive anterior wrist and proximal thumb. In vivo analyses with synthetic enhancers, in which human-specific substitutions were introduced into the chimpanzee enhancer sequence or reverted in the human enhancer to the ancestral state, indicated that 13 substitutions clustered in an 81-base pair module otherwise highly constrained among terrestrial vertebrates were sufficient to confer the human-specific limb expression domain.

As to the scientific vacuity of Intelligent Design, ask yourself the following simple question: How does Intelligent Design explain these findings? Is it anywhere similar to how Dembski 'argued'?

Dembski wrote:

> As for your example, I'm not going to take the bait. You're asking me to play a game: "Provide as much detail in terms of possible causal mechanisms for your ID position as I do for my Darwinian position." ID is not a mechanistic theory, and it's not ID's task to match your pathetic level of detail in telling mechanistic stories. If ID is correct and an intelligence is responsible and indispensable for certain structures, then it makes no sense to try to ape your method of connecting the dots. True, there may be dots to be connected. But there may also be fundamental discontinuities, and with IC systems that is what ID is discovering.


William A. Dembski Organisms using GAs vs. Organisms being built by GAs thread at ISCID 18. September 2002

Michael Eisen describes his views on the [Scientists cynical use of "Junk DNA"](http://www.michaeleisen.org/blog/?p=4), if only ID proponents could show a similar skepticism.

Finding regulatory elements is quite a bit more complex than finding genes, and only recently the computational power has allowed science to more accurately study regulatory elements. In this case

John Timmer wrote:

> The recent paper focuses on a sequence, called HACNS1, which is about 550 bases long. If it were picking up mutations at random, the human version would be expected to have four; instead, it has 13 differences with the chimp sequence.

In other words, science found how a sequence had more than the expected number of differences between humans and chimps. Such an observation requires an explanation. 

This lead to the conclusion

John Timmer wrote:

> HACNS1 lies outside of any known genes, suggesting it is regulatory DNA. The researchers hooked it up to a gene that couldn't otherwise be expressed and injected the DNA into mice. As expected, the regulatory function caused the gene to be expressed in a very specific pattern in the head and limbs. The key result came when the chimp version was hooked up to the same gene and injected into mice--the limb expression was severely reduced or absent.
> 
> The obvious inference here is that the sequence drives human-specific gene expression, specifically in the areas that form the hands and feet, which are obviously quite distinct in humans. The researchers also made two constructs that had only six of the 13 changes from chip to humans and found that these drove expression was somewhere in between the humans and the chimp sequences, which is exactly what you'd expect from a gradual, evolutionary change.
> 
> There's still work to be done, as HACNS1 lies about halfway between two different genes, and the researchers don't know which of the two it regulates. Still, it's a pretty exciting result, and worthy of the attention it received. 

And that is how real science works.
