---
layout: article
title: Flagellum evolution paper exhibits canine qualities
date: '2007-04-16 22:49:19 -0700'
author: Nick Matzke
mt_id: 2973
blog_id: 2
post_id: 2973
basename: flagellum_evolu_1
---
Today the prestigious journal _Proceedings of the National Academy of Sciences_ (_PNAS_) put out an Advanced Online Publication paper on flagellum evolution entitled, "[Stepwise formation of the bacterial flagellar system](http://www.pnas.org/cgi/content/abstract/0700266104v1)."  The paper is [freely available](http://www.pnas.org/cgi/content/abstract/0700266104v1) via Open Access.  I was initially excited that _PNAS_ had published a paper on this topic, and furthermore that it cited the [Pallen/Matzke essay on flagellum evolution](/archives/2006/09/flagellum-evolu.html), and Ian Musgrave's excellent book chapter in [_Why Intelligent Design Fails_](http://www2.truman.edu/~edis/books/id/).  

Unfortunately, as I read the paper, my delight turned to concern, and then dismay.  The paper makes some potentially useful points and explores new territory in a few areas.  But much of it ranges from dubious to just irremediably wrong.

I am not talking about minor issues, like the fact that the authors endorse the "flagellum first, type 3 secretion derived" position, which is currently debated by the experts.  I am talking about things like the conclusion of the paper, which is that

> "we have shown the bacterial flagellum too originated from 'so simple a beginning,' in this case, a single gene that underwent successive duplications and subsequent diversification during the evolution of Bacteria."

The authors argue, as have I and others, that the flagellar axial proteins, about 10 of them, evolved by duplication and diversification from a common ancestor.  But this is miles away from demonstrating that _all_ flagellar proteins come from _one_ gene, and in fact the extra-flagellar homologies show positively that this _cannot_ be true, as anyone who has read the literature on this topic should know (the authors exhibit some awareness of these homologies, but they don't seem to see the implications).  Frankly it is a flabbergasting thing to say, and I can't understand how it got published.

There are numerous other detailed problems, especially involving the statistics of homology searches, and the authors' truly weird interpretation of their phylogeny of the axial proteins (the phylogenies are in the Supplementary Figures).  They all seem to point to unfamiliarity with the basic issues involved in these areas -- and if I can perceive this as an amateur, you know something is wrong.  To give this paper a pass as it is, the peer-reviewers must have been some combination of not in the right fields, too busy to do a serious review, etc.

Normally I would just ignore this sort of paper.  Weird stuff gets published in obscure journals and ignored all the time.  But it is published in _PNAS_, and _PNAS_ has put it up [free on the web](http://www.pnas.org/cgi/content/abstract/0700266104v1), and a few pro-evolution blogs have already linked to it ([1](http://interrogatingnature.blogspot.com/2007/04/stepwise-formation-of-bacterial.html), [2](http://evilutionarybiologist.blogspot.com/2007/04/irreducible-complexity-indeed.html) -- sorry guys), and sooner or later the creationists will realize that the people who know something about flagellum evolution have major problems with the paper and start to make hay of it.  And if we don't stomp on this paper, more people who don't know their flagella from their cilia will cite the _PNAS_ piece as a competent rebuttal to Behe and co.  So as painful as it is, and as much as I hate sticking my neck out against established scientists of good reputation, people who clearly could have produced a good paper with some more rigorous reviewing and background research, there is nothing for it but to suck it up and declare this paper a dog.  In science, if the choice is between propagating error and being remorselessly negative, you've got to go with remorselessness. It sucks, and it's nothing personal -- everyone makes mistakes sometimes -- but that's science (unlike [creationism](/archives/2007/04/john-mark-reyno.html)/[ID](/archives/2006/11/casey-luskins-s.html)).

Am I being too harsh?  It's always possible that my inexpert self has missed something key that the experts are aware of.  Read the paper, read the other [flagellum evolution posts](http://www.pandasthumb.org/archives/irreducible_complexity/flagellum_evolution/) and [papers](http://www.talkdesign.org/faqs/flagellum.html#update) (and [more papers](http://www.talkdesign.org/faqs/flagellum_background.html#rebuttals)), and give your opinion.  It's even possible that we are accidentally seeing a rough draft or something.

PS: It would take all night to write a completely coherent analysis, but I should provide some details on just what the problems are that have me annoyed.  Here is some initial analysis I made on email, after someone emailed me for comment (edited to remove specific people etc.):

_\[Note on terminology: the "axial proteins" make up the rod, hook, and filament of the flagellum.  Together they form one long tube that constitutes most of what you see when you look at a flagellum.  However a dozen other non-axial proteins secrete the axial proteins, provide the motor power, etc.\]_

> \[I was asked for reaction, and I replied as follows\]
> 
> **Positive points in the paper:**
> 
> \* (FliC + FlgL) are homologous to the (rod-hook proteins) on their analysis.  I and I suspect everyone who thought about it always thought this was extremely likely, but AFAIK it had not been explicitly claimed in peer-reviewed publications, and it wasn't claimed in the Pallen-Matzke paper, where we just traced the axial proteins back to 2 ancestral proteins.  Whether or not the evidence is good enough to be considered demonstrated in a peer-reviewed journal is the real question.  
> 
> \* Identification of 24 ancestral core proteins, which we did not quite make explicit (but then I am dubious about a few of their decisions, see below)
> 
> \* The comparison of flagellar systems to bacterial phylogeny is also worthwhile although their basic conclusion (mostly linear inheritance of systems, but with some lateral transfer) has been published before.
> 
> \* Suggestion of homology of (unnamed???) flagellar proteins to Type V secretion, the P pilus, and prophage tail proteins.  These would all be interesting if true.  However, if it just the ATPase protein (which has endless homologs), or the muramidase domain of FlgJ, or other known bits and pieces, it is not new.  
> 
> On the other hand, it is easy to get false positives in these searches for weak similarities, especially with certain domains that are widespread.  These domains might share common ancestry or they might conceivably converge; and even if they do indicate common ancestry it might be from the evolution of the protein repetoire during the origin of prokaryotes, rather than any events occurring specifically with the origin of the flagellum.
> 
> **Things I am dubious about:**
> 
> \* I have not delved into their methods in detail, but it looks like this was basically another exercise in BLASTing.  They seem to be proposing a bunch of new homologies (see Figure 3, grey lines) based on extremely thin evidence.  The homologies proposed between the various membrane proteins would be very important if true, but membrane proteins exhibit certain sequence patterns just because they are membrane proteins.  
> 
> I tend to think some flagellum membrane proteins are homologous to some F1Fo-ATPase membrane proteins, but thinking it is different than proving it.
> 
> \* They assert their preference for the flagellum-ancestral-to-NF-T3SS model, but don't deal with the counterarguments or add any new data.  To be fair, the sister-groups people need to produce better evidence on their side also.
> 
> \* They don't cite the Pallen et al. 2006 paper in Protein Science, documenting homology between FliH and ATPase components, even though it is extremely relevant to their topic and was cited by the structure people in recent papers.
> 
> \* Asserting that the L- and P-ring proteins were not part of the common ancestor, based on the idea that gram-positive bacteria and spirochetes are phylogenetically basal, is fine, but it is no stronger than your confidence in the branching structure of the bacteria phyla, which for me is not very confident and plus depends upon the rooting of the bacterial tree.
> 
> \* They claim their model is step-by-step, but it is less step-by-step than the Pallen-Matzke paper AFAICT.
> 
> \* The paper could be repaired by removing the dubious points, but then there would not be much new about it, although the FliC-hook homology would be worth publishing by itself if they have a convincing case. 
> 
> **Overall (but very preliminary) conclusion:**
> 
> \* Confirms the basic results of previous studies, makes a few speculative suggestions and takes sides on a few issues of opinion, but mostly indicates that certain outstanding questions need to be worked on, e.g. the phylogenetic relationship of F-T3SS and NF-T3SS.
> 
> **\[after a more detailed read\]**
> 
> \* The supplementary material has a useful attempt at a phylogeny of the axial proteins, and it is at least good to have someone making an explicit hypothesis -- but the bootstrap values for the phylogeny are almost all 75% and below, i.e. not good.  And the stuff about maybe all the membrane proteins also originating from duplication/divergence is not supported at all as far as I can tell.
> 
> \* OK, I went and had a careful read-through.  The paper is:
> 
> 1. A perhaps-worthwhile comparison of flagella phylogenies to bacterial phylogenies
> 
> 2. A perhaps-useful attempt to analyze the phylogeny of the axial proteins
> 
> 3. But even if #1 and #2 are done right, this is hidden within lots of totally mystifying rhetoric stating that #2 explains the whole flagellum in step-by-step fashion, even though the axial proteins are really only about 1/2 of the "flagellar core".  And with many dubious interpretations on specific points.
> 
> The most mystifying sentence is this one.  It is the last sentence of the conclusion:
> 
> > As with the evolution of other complex structures and processes (29--32), we have shown the bacterial flagellum too originated from "so simple a beginning," in this case, a single gene that underwent successive duplications and subsequent diversification during the early evolution of Bacteria.
> 
> There is just no way that the flagellum evolved by diversification of a single gene.  If they meant that just the axial proteins (rod, hook, flagellin) originated from a common ancestor, that would be fine, but they seem to say the whole flagellum, based on a very muddled argument earlier that "we conclude that despite their antiquity, the similarities among core proteins to one another are more common and, on average, stronger than to nonflagellar proteins." As if "on average" was relevant when it could be true (probably is) that half the flagellar proteins originated by internal duplications and the other half by cooption.
> 
> \[...\]
> 
> **\[talking about the interpretation of the axial protein phylogeny\]**
> 
> Yeah, I was going to mention the polarity problem also \[with the interpretation of the phylogeny of the axial proteins\].  Even if you assume the root is between (FliC+relatives) and (rod+hook) proteins, which IMHO is reasonable if not proven, and even if you accept the phylogeny despite the low confidence numbers (and it might be a reasonable phylogeny of the axial proteins), that \*still\* doesn't tell you that FlgB/FlgC emerged first.  What it would tell you is the first thing that happened is the divergence of the root into a proto-"flagellin" group and a proto-"rod/hook" group.
> 
> I think they get the bit about FlgB/FlgC coming first from the bottom of Supplementary Figure A, where that divergence is furthest to the left (by a tiny, tiny bit that is surely not statistically significant).
