---
layout: article
title: You have to spend money to make money
date: '2007-10-09 22:34:21 -0700'
author: Arthur Hunt
mt_id: 3287
blog_id: 2
post_id: 3287
basename: you_have_to_spe
---
In biology, genetics, biochemistry, and molecular biology classes, one of the things that we used to learn that distinguishes prokaryotes from eukaryotes is the “fact” that eukaryotes have polyadenylated mRNAs, while prokaryotes do not.  This morphed rather easily into a distinction – eukaryotes do polyadenylation, prokaryotes do not.  For years, this was standard fare in class.  However, even as generations of students (beginning with the discovery of polyadenylate tracts in hnRNA in eukaryotes) were learning of this distinction, we knew that all was not right with this.  Among the lurking pieces of conflicting data was that the first biochemical entity that was shown to add poly(A) tracts to RNAs in vitro was a bacterial one, isolated and purified from E. coli (1).

Slowly but surely, clarity was brought to this backwater of molecular biology.  In the 1980’s, Nilima Sarkar and her coworkers championed the idea that bacteria do polyadenylate RNAs, and published a series of studies that eventually caught the eyes of other prokaryotic molecular biologists (2).  Sarkar labored under the assumption (that, if one reflects on the matter, derives from a combination of teleological logic, historical accident, and evolutionary reasoning) that polyadenylation in prokaryotes would have the same function as had been established in eukaryotes – namely, poly(A) stabilizes RNAs and promotes their translation (3).  Others picked up this work and followed a different trail.  The experiments are too numerous to list in this essay, but the conclusion they lead to is easy to summarize – in prokaryotes, polyadenylation is a part of the process by which RNAs are broken down.  Briefly, polyadenylation is associated with the so-called degradosome, aka the multi-enzyme complex that is the RNA “garbage disposal” of the prokaryotic cell (4).  Indeed, not only is the previously-mentioned poly(A) polymerase associated (at least in some cases) with the degradosome, it turns out that the central enzyme of the degradosome, polynucleotide phosphorylase, in fact can function in vivo (as well as in vitro) as a poly(A) polymerase (5).  While all of the mechanistic details are not yet with us, a fascinating general picture of the role of polyadenylation in the process of RNA degradation is available.  Again being brief, the addition of a poly(A) tail to an RNA is a means by which its degradation by the degradosome is promoted.  Some look upon the poly(A) tract as a sort of signal, and it has been proposed that polyadenylation is a mechanism for “kick-starting” the complex of 3’-&gt;5’ exonucleases\* that have stalled at places along the RNA that are structured (6).  The association with the turnover of structured RNA is particularly relevant to the observation that stable RNAs (e.g., ribosomal RNAs) may be polyadenylated in bacteria (7).

This would make a good “stopping point” for what would be a short essay, one that makes an interesting point that is, IMO, rather counterintuitive and contrary to simple teleological reasoning – in prokaryotes, it is necessary to synthesize RNA (polyA) solely for the purpose of making more efficient the turnover of RNA.  (It pays to note that the processes of turnover and polyadenylation are both vital to the cell, as mutants that lack all of the associated exonucleases, as well as cells that are deficient in all of the poly(A) polymerases, are very slow-growing or dead.)  But the essay doesn’t end with this note.  As mentioned previously, one of the ideas bouncing around in the 1980’s and early 1990’s regarding the polyadenylation of RNA in prokaryotes was that the poly(A) tract functioned in prokaryotes as it does in eukaryotes.  This seemed to be a reasonable conjecture, partly because one might expect organisms with a shared ancestry would have preserved the functionality of such a basic entity as poly(A) throughout the course of evolutionary history.  However, a great deal of research has painted a different picture, one that is likely to be encountered in textbooks today – in eukaryotes, poly(A) stabilizes RNA and promotes translation while in prokaryotes, poly(A) destabilizes RNA.  

But is this picture completely accurate?  After all, eukaryotes possess an RNA “garbage disposal” that is quite similar to the degradosome.  The eukaryotic complex has been termed the exosome, and is a multi-enzyme complex that consists of a number of RNA helicases as well as a set of 3’-&gt;5’ exonucleases, including several that share both sequence similarity and an important biochemical property with polynucleotide phosphorylase.  (The biochemical similarity is the ability to add, instead of water, inorganic phosphate to the broken phosphodiester bond, producing NDPs instead of NMPs in the course of breakdown.  The reverse reaction, adding adenylate moieties to the 3’ ends of RNAs using ADP as a substrate, is probably the basis for the poly(A) polymerase activity of the enzyme in vivo.  Also, it’s a curious historical sidelight to note that polynucleotide phosphorylase was, to this author’s knowledge, the first such polymerase discovered, and was touted for a time as _the_ RNA polymerase \[8\].)

It turns out (happily enough for this series) that the answer to this question is “no”.  As early as 2000 (and probably earlier), it was known that there was a genetic connection between poly(A) polymerase and a subunit of the nucleus-localized “version” of the exosome in yeast (9).  A spate of later publications demonstrated a number of connections between the polyadenylation machinery and the exosome, and this line of research culminated, in recent reports, with the demonstration that, as is the case in prokaryotes, polyadenylation is associated with, and part of the functioning of, the eukaryotic exosome (10).  This activity has been proposed to constitute (at least part of) a sort of quality-control mechanism, whereby RNAs that are made by pol II but are for some (to this author’s mind unknown) reason aberrant are targeted or channeled to the exosome instead of the RNA processing and transport pathway.  It is also involved in the degradation of (presumably damaged or nonfunctional) ribosomal RNA and transfer RNA.

There are many fascinating ramifications of these findings, and most of these must sadly remain beyond the scope of this series.  But some are relevant.  One is the realization that the evolutionarily-conserved (mayhaps even primordial) role for the poly(A) tail in living things is one of turnover and breakdown.  A second is the corollary of the first, that the roles for poly(A) that most students learn – stabilizing mRNA and promoting translation – are secondary and derived.  Lastly, the function of poly(A) in turnover reveals a decidedly counterintuitive process – it is necessary to make RNA in order to break it down.

References and footnotes

1. Oasa S, Tsugita A, Mii S. 1972. Nat New Biol. 240(97): 39-41. 

2. Gopalakrishna Y, Langley D, Sarkar N. 1981. Nucleic Acids Res. 9(14): 3545-54.
Gopalakrishna Y, Sarkar N. 1982. Biochemistry 21(11): 2724-9. 
Gopalakrishna Y, Sarkar N. 1983. Arch Biochem Biophys. 224(1): 196-205.

3. Kalapos MP, Paulus H, Sarkar N. 1997. Biochimie. 79(8): 493-502.

4. Carpousis AJ, Vanzo NF, Raynal LC. 1999. Trends Genet. 15(1): 24-8. 
Steege DA. 2000. RNA6(8): 1079-90. 

5. Mohanty BK, Kushner SR. 2000. Proc Natl Acad Sci U S A. 97(22): 11966-71. 
Yehudai-Resheff S, Hirsh M, Schuster G. 2001. Mol Cell Biol. 21(16): 5408-16. 

6. Blum E, Carpousis AJ, Higgins CF. 1999. J Biol Chem. 274(7): 4009-16.

7. Li Z, Pandit S, Deutscher MP.  1998. Proc Natl Acad Sci U S A. 95(21): 12158-62. 

8. Grunberg-Manago M, Ortiz PJ, Ochoa S. 1955. Science 122: 907-910.
\[Another interesting aside – Grunberg-Manaao and Ochoa would win the Nobel prize for work performed using PNPase as a tool for preparing nucleic acids of defined composition, thus providing a way to determine the “direction” of the genetic code as well as specific amino acid-codon correspondences.\]

9. Burkard KT, Butler JS. 2000. Mol Cell Biol. 20(2): 604-16. 

10. Wyers F, Rougemaille M, Badis G, Rousselle JC, Dufour ME, Boulay J, Régnault B, Devaux F, Namane A, Séraphin B, Libri D, Jacquier A. 2005. Cell. 121(5): 725-37. 
LaCava J, Houseley J, Saveanu C, Petfalski E, Thompson E, Jacquier A, Tollervey D. 2005. Cell 121(5): 713-24.
Vanácová S, Wolf J, Martin G, Blank D, Dettwiler S, Friedlein A, Langen H, Keith G, Keller W. 2005. PLoS Biol. 3(6):e189.
Egecioglu DE, Henras AK, Chanfreau GF. 2006. RNA 12(1): 26-32. 
\[The paper by Wyers et al. will be the subject of a subsequent essay on The Panda’s Thumb, one that discusses the relationship between RNA turnover and the existence of transcripts encoded by "junk DNA".\]

\* - 3’-&gt;5’ denotes the direction of “movement” of a nuclease (or polymerase, for that matter) along a polynucleotide.  RNA or DNA chains have a directionality that is defined by their extremes.  At one end, the 5’-carbon group of the ribose ring is not attached to an adjacent nucleotide; this end is called, naturally enough, the 5’ end.  Likewise, at the other end, the 3’ carbon of the ribose ring is not attached to an adjacent nucleotide.  Thus, a 3’-&gt;5’ exonuclease breaks RNA down beginning with the 3’ end of the molecule.  As a rule, 5’-ends have phosphates (or, if you’re a eukaryotic mRNA, more elaborate “caps”) attached to the hydroxyl moieties on the 5’-carbon, while 3’-ends have free hydroxyl groups.  These chemical properties are important for polymerases and nucleases.
