---
layout: article
title: 'Alternative routes and mutational robustness in complex regulatory networks '
date: '2007-08-22 11:52:29 -0700'
author: PvM
mt_id: 1403
blog_id: 2
post_id: 1403
basename: alternative_rou
---
Although some have called systems biology a 'friend of Intelligent Design', reality is that systems biology is all but a friend of what is best known as 'ignorance'.

In a recent article in BioSystems 88 (2007) 163â€“172, titled "Alternative routes and mutational robustness in complex regulatory networks", Andreas Wagner and Jeremiah White describe how 

> Alternative pathways through a gene regulation network connect a regulatory molecule to its (indirect) regulatory target via  different intermediate regulators. We here show for two large transcriptional regulation networks, and for 15 different signal transduction networks, that multiple alternative pathways between regulator and target pairs are the rule rather than the exception. We find that in the yeast transcriptional regulation network intermediate regulators that are part of many alternative pathways between a regulator and target pair evolve at faster rates. This variation is not solely explicable by higher expression levels of such  regulators, nor is it solely explicable by their variable usage in different physiological or environmental conditions, as indicated by  their variable expression. This suggests that such pathways can continue to function despite amino acid changes that may impair one intermediate regulator. Our results underscore the importance of systems biology approaches to understand functional and evolutionary constraints on genes and proteins.

So while ID proponents are arguing for an 'edge' to evolution, real science is uncovering a remarkable richness for evolution.

So let me ask the following question: Who has contributed to scientific knowledge here?

The question now becomes, how can such robustness evolve?

The answer, not surprisingly, is via Darwinian pathways...

> The topology of cellular circuits (the who-interacts-with-whom) is key to understand their robustness to both mutations and noise. The reason is that many biochemical parameters driving circuit behavior vary extensively and are thus not fine-tuned. Existing work in this area asks to what extent the function of any one given circuit is robust. But is high robustness truly remarkable, or would it be expected for many circuits of similar topology? And how can high robustness come about through gradual Darwinian evolution that changes circuit topology gradually, one interaction at a time? We here ask these questions for a model of transcriptional regulation networks, in which we explore millions of different network topologies. Robustness to mutations and noise are correlated in these networks. They show a skewed distribution, with a very small number of networks being vastly more robust than the rest. All networks that attain a given gene expression state can be organized into a graph whose nodes are networks that differ in their topology. Remarkably, this graph is connected and can be easily traversed by gradual changes of network topologies. Thus, robustness is an evolvable property. This connectedness and evolvability of robust networks may be a general organizational principle of biological networks. In addition, it exists also for RNA and protein structures, and may thus be a general organizational principle of all biological systems.

Ciliberti, S, Martin, OC, Wagner, A. (2007) Robustness can evolve gradually in complex regulatory networks with varying topology. PLoS Computational Biology 3(2): e15.
