---
layout: article
title: How to examine the evolution of proteins
date: '2011-10-20 14:05:44 -0700'
author: PZ Myers
mt_id: 6374
blog_id: 2
post_id: 6374
basename: how_to_examine
---
<p class="lead">In my previous post, I described <a href="http://freethoughtblogs.com/pharyngula/2011/10/20/how-not-to-examine-the-evolution-of-proteins/">the misguided approach Gauger and Axe have taken to criticizing evolution</a>, and one of the peculiarities of their criticism is that they cited another paper by <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3116920/?tool=pubmed">a paper by Carroll, Ortlund, and Thornton</a> which traced (successfully) the evolutionary history of a class of proteins. Big mistake. As I pointed out, one of the failings of the Gauger/Axe approach is that they're asking how one protein evolved into a cousin protein, without considering the ancestral history &hellip;they make the error of trying to argue that an extant protein couldn't have directly evolved into another extant protein, when no one argues that they did.</p>

<p>The tactical error is that right there in the <i>very first paragraph</i> of their paper, Carroll, Ortlund, and Thornton point out the fallacy of what the creationists were doing.</p>

<blockquote><p>Direct comparisons among present-day proteins can sometime yield insights into the sequence and structural mechanisms that underlie functional differences. Such "horizontal" comparisons, however, cannot determine which protein features are ancestral and which are derived, so they are not suited to reconstructing the events that produced functional diversity.</p></blockquote>

<p>They don't mention Gauger and Axe, of course &mdash; this paper was written <i>before</i> the creationists wrote theirs &mdash; but a methodological flaw is still spelled out plainly, the creationists reference it so I presume they read it, and they still charged ahead and did their flawed study, and then had the gall to claim their work was superior.</p>


<!--more-->


<p>Ah, silly creationists. They just assume their target audience won't bother to read the work they're citing, and isn't competent to understand it anyway. And they're usually right.</p>


<p>The crew doing the work in the Carroll paper did not make the same mistakes. They are doing ancestral sequence reconstruction (ASR), so the effort to work backward to trace ancestral states is implicit. The bulk of the paper describes the sequencing of homologous and paralogous genes in more organisms (in this case, especially cartilaginous fishes), and the analysis of synthesized, reconstructed ancestral proteins, so it's built entirely on an empirical foundation. And their answers actually advance our understanding of the base-by-base changes that led to the evolution of the current set of proteins. I think they were courteous and sensible (and probably, the idea didn't even occur to them) in not comparing their work to that of the creationists &mdash; it would have been less than gracious to point out how ugly, cheap, and cheesy the stuff coming out of the Biologic Institute looks.</p>

<p>What the real scientists were studying is a class of receptors that respond to mineralocorticoid and/or glucocorticoid hormones. These proteins are similar in sequence and structure to one another, and are clearly paralogous: they arose by an ancient gene duplication event, somewhere around 450 million years ago. The two copies have since diverged to have different roles in hormone physiology.</p>

<p>The two receptors are called MR, for mineralocorticoid receptor, and GR, for glucocorticoid receptor.</p>

<p><b>MR</b>s are activated by adrenal hormones, aldosterone and deoxycorticosterone, and to a lesser exent, cortisol. The receptors are extremely sensitive to the hormones. These hormones are important in regulating salt balance, and you might well imagine that in our fishy ancestors, as well as ourselves, regulating the concentrations of salts in our blood and tissues is a very important function. Deviations can cause death, after all.</p>

<p><b>GR</b>s are activated by high doses of cortisol; these receptors are much less sensitive, requiring high doses of the hormone to trigger a response. They are important in regulating stress responses: they adjust the immune system and sugar metabolism. These aren't 'twitchy', fast response functions like maintaining salt balance is; they are long-term, 'last-ditch' reactions to growing stresses, so functionally it makes sense that activation requires high levels of accumulated hormone.</p>

<p>Using ASR techniques &mdash; phylogenetic analysis and estimating the most likely sequence of the ancestral protein &mdash; the investigators have put together a picture of the receptor before MR and GR diverged. This protein is called <b>AncCR</b>, for Ancestral Corticosteroid Receptor, and it has been synthesized in the lab, so we know about its properties. AncCR is a lot like MR: it's sensitive to low concentrations of hormone, and it responds to low concentrations of a broad spectrum of hormones.</p>

<p>The pedigree of these proteins is illustrated below.</p>

<div style="text-align: center; font-size: 11px"><a href="http://scienceblogs.com/pharyngula/upload/2011/10/grmr_phylo.php" onclick="window.open('http://scienceblogs.com/pharyngula/upload/2011/10/grmr_phylo.php', 'popup', 'width=910,height=510,scrollbars=no,resizable=no,toolbar=no,directories=no,location=no,menubar=no,status=no,left=0,top=0' ); return false"><img src="http://scienceblogs.com/pharyngula/upload/2011/10/grmr_phylo-thumb-500x280-70023.gif" width="500" height="280" alt="grmr_phylo.gif"/><br />(Click for larger image)</a><br /><b>Simplified phylogeny of corticosteroid receptors.</b> Ancestral sequences are shown at relevant nodes: AncCR, the last common ancestor of all MRs and GRs; AncGR1, the GR ancestor of cartilaginous fishes and bony vertebrates; AncGR2, the GR ancestor of ray- and lobe-finned fishes (including tetrapods); AncMR1, the MR ancestor of cartilaginous fishes and bony vertebrates. (AncGR1.0 and AncGR1.1 are different reconstructions of node AncGR1, inferred from datasets with different taxon sampling.) Black, high sensitivity receptors; gray, low sensitivity receptors. Single and double gray dashes mark functional shifts towards reduced sensitivity and increased specificity, respectively. Support values are the chi-square statistic (1 - p, where p equals the estimated probability that a node could occur by chance alone) calculated from approximate likelihood ratios. The length of branches from AncCR to AncMR1 and to AncGR1, expressed as the mean number of substitutions per site, are indicated in parentheses.</div>

<p>The MRs are similar in function to the AncCR, so they aren't particularly interesting in this context &mdash; there's no big question about how the MRs retained similar properties to their ancestor. The interesting questions are all about the GRs: what changed to make GRs <i>different</i> from the ancestral protein? What amino acid changes set AncGR1 apart from AncCR?</p>

<p>The investigators have an answer. The first step was the evolution of <i>reduced</i> hormone sensitivity, so that these receptors only responded to very high concentrations of the hormone, and the second step was a loss of sensitivity to the mineralocorticoids, already handled by the MRs, so that they only respond to high doses of cortisol, which at this point became exclusively a stress hormone. And they know <i>exactly</i> which amino acids changed to confer the reduced sensitivity.</p>

<p>They identified three changes: the conversion of a valine at position 43 into an alanine, called V43A; the conversion of an arginine at position 116 into a histidine, R116H; and the conversion of a cysteine at position 71 into a serine, C71S. They also know the effect of the mutations. V43A and R116H each loosen the structure of the receptor so that it's less sensitive, and when both mutations are present the effect greatly reduces sensitivity about 10,000-fold&hellip;too much! They make the mutant hormone <i>too</i> insensitive, and much less insensitive than their reconstructed AncGR1.</p>

<p>The most interesting change is C71S. It basically does nothing to the sensitivity; make the C71S change to AncCR, and you get a receptor protein that is essentially indistinguishable in its response. This is effectively a <b>neutral</b> mutation. It can spread freely through a population with no deleterious or advantageous effect.</p>

<p>C71S does have one significant effect in cooperation with the other two mutations: it buffers both V43A and R116H. When all three mutations are present, the desensitizing effects of V43A and R116H are reduced to produce the level of sensitivity expected for the AncGR1 protein. This means we can reconstruct the order of the amino acid changes in evolution. First came C71S, because it doesn't cause any particular adaptive change, and because if either V43A or R116H came first, the resulting receptor would be generally non-functional. The existence of C71S first means the subsequent V43A/R116H changes produced receptors that are still functional, but simply operate only at higher concentrations of the hormones.</p>

<p>All of these changes are perfectly compatible with an evolutionary model of their origin. No sudden leaps, no deleterious intermediates are required &mdash; everything hangs together beautifully and is backed up by solid empirical evidence. In addition, the work explains the mechanics of receptor-hormone interactions, stuff I haven't explained here, but if you're a biochemist, there's much to savor in the paper.</p>

<p>It's an amazing contrast to the <a href="http://freethoughtblogs.com/pharyngula/2011/10/20/how-not-to-examine-the-evolution-of-proteins/">Gauger and Axe paper</a>, too. No wonder I'm not a creationist!</p>


<hr width="25%" align="left" />
<p class="ref">Carroll SM, Ortlund EA, Thornton JW (2011) Mechanisms for the evolution of a derived function in the ancestral glucocorticoid receptor. PLoS Genet.7(6):e1002117. Epub 2011 Jun 16.</p>

