---
layout: article
title: Flagellum evolution in Nature Reviews Microbiology
date: '2006-09-07 04:29:30 -0700'
author: Nick Matzke
mt_id: 2480
blog_id: 2
post_id: 2480
basename: flagellum_evolu
---
When I returned from vacation today, I was surprised to discover this new article pop up in my automated searches for  flagellum stuff in the literature databases:

> Pallen MJ, Matzke NJ. (2006). "[From _The Origin of Species_ to the origin of bacterial flagella](http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html)." _Nature Reviews Microbiology_, 4(10), 784-790. October 2006.  Advanced Online Publication on September 5, 2006. \[[PubMed](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16953248&amp;query_hl=4&amp;itool=pubmed_docsum)\] \[[Journal](http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html)\] \[[DOI](http://dx.doi.org/10.1038/nrmicro1493)\] \[[Google Scholar](http://scholar.google.com/scholar?q=From+The+Origin+of+Species+to+the+origin+of+bacterial+flagella&amp;ie=UTF-8&amp;oe=UTF-8&amp;hl=en&amp;btnG=Search)\]
> 
> In the recent Dover trial, and elsewhere, the 'Intelligent Design' movement has championed the bacterial flagellum as an irreducibly complex system that, it is claimed, could not have evolved through natural selection. Here we explore the arguments in favour of viewing bacterial flagella as evolved, rather than designed, entities. We dismiss the need for any great conceptual leaps in creating a model of flagellar evolution and speculate as to how an experimental programme focused on this topic might look.

[Matzke NJ](http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Search&amp;itool=pubmed_AbstractPlus&amp;term=%22Matzke+NJ%22%5BAuthor%5D)?  Hey, I know that guy!  

I thought that this article was not coming out until October, and I would therefore have a few weeks to prepare some suitable PT posts to update everyone on developments in flagellar evolution since [my first effort in 2003](http://www.talkdesign.org/faqs/flagellum.html) (read [this](http://www.talkdesign.org/faqs/flagellum_background.html) for background), and on the litany of errors and pseudoscience that the ID movement has produced regarding their favorite "Icon of Intelligent Design."  But, the powers that be at _Nature Reviews Microbiology_ have seen fit to [release the article as an Advanced Online Publication](http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html) -- and put it on their [front page](http://www.nature.com/nrmicro/index.html), no less -- so it remains a secret no longer.  _C'est la vie_.

There are several large flagellum-related topics I still plan to blog in the near future, but for the moment I would just like to hit one topic: what is The Main Point that PT readers should get out of the article?

**The Main Point**

A number of issues are discussed in this paper.  I won't belabor most of them (read the paper!), but I do want to make sure one key point is well understood by everyone in the evolution vs. ID/creationism debate -- or at least among us skeptics of ID, since the ID/creationists probably won't listen.

Referring to the Kitzmiller case, we note in [Box 2: Of forelimbs and flagella](http://www.nature.com/nrmicro/journal/v4/n10/box/nrmicro1493_BX2.html),

> ID advocates say that their position is supported by discontinuities between the flagellum and the rest of the biological world, just as a designed entity like a watch differs from an undesigned entity, such as a stone. In support of this line of reasoning, Scott Minnich in his expert witness report claimed that "the other thirty proteins in the flagellar motor (that are not present in the type III secretion system) are unique to the motor and are not found in any other living system." As our discussion shows, this is not true. Instead, we have detected sequence homologies linking flagellar components to the rest of the biological universe (Table 1).
> 
> (Pallen and Matzke 2006, "[From _The Origin of Species_ to the origin of bacterial flagella](http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html)." _Nature Reviews Microbiology_ advance online publication 5 September 2006.)

Below, I will show that Minnich's claim has been repeated by many other leading ID advocates, and that they have all got their basic facts badly wrong.

**_What ID advocates have claimed_**

First, let's look at what the IDists have written while arguing against the evolution of the flagellum:

> It follows that the TTSS does not explain the evolution of the flagellum (despite the handwaving of Aizawa 2001). Nor, for that matter, does the bacterial flagellum explain in any meaningful sense the evolution of the TTSS. The TTSS is after all much simpler than the flagellum. The TTSS contains ten or so proteins that are homologous to proteins in the flagellum. The flagellum requires an additional thirty or forty proteins, which are unique.
> 
> William A. Dembski (2003). "[Still Spinning Just Fine: A Response to Ken Miller](http://www.designinference.com/documents/2003.02.Miller_Response.htm)." DesignInference.com. February 17, 2003.

> With the bacterial flagellum, you're talking about a machine that's got 40 structural parts.  Yes, we find 10 of them are involved in another molecular machine, but the other 30 are unique!  So where are you going to borrow them from?  Eventually you're going to have to account for the function of every single part as originally having some other purpose.   So you can only follow that argument so far until you run into the problem of you're borrowing parts from nothing.
> 
> Scott Minnich (2003), in the video _Unlocking the Mystery of Life_, [online at The Apologia Project](http://www.theapologiaproject.org/video_library.htm).

> Miller's scenario faces at least key three difficulties. First, the other thirty or so proteins in the flagellar motor are unique to it and are not found in any other living system. From where,then, were these protein parts co-opted?
> 
> Stephen C. Meyer (2004). "[Verdict on the Bacterial Flagellum Premature: A Response to Begley's 'Evolution Critics Come Under Fire...' in the _Wall Street Journal_](http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=1843)."  Discovery Institute website, February 19, 2004.

> Additionally, the other thirty proteins in the flagellar motor (that are not present in the TTSS) are unique to the motor and are not found in any other living system. From whence, then, were these protein parts co-opted? 
> 
> [Minnich (2005) expert report](http://www2.ncseweb.org/kvd/index.php?path=experts/), March 31, 2005 / Scott A. Minnich & Stephen C. Meyer (2004). "[Genetic Analysis of Coordinate Flagellar and Type III Regulatory Circuits in Pathogenic Bacteria](http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;program=CSC%20-%20Scientific%20Research%20and%20Scholarship%20-%20Science&amp;id=2181)." Second International Conference on Design & Nature, Rhodes Greece.  Wessex Institute of Technology, September 1, 2004.

> With regards to the flagellum at least 2/3 of the parts are not known to be shared with any other structure therefore might not be even a sub-part of another system at all.
> 
> Casey Luskin (2006). "[Do Car Engines Run on Lugnuts? A Response to Ken Miller & Judge Jones's Straw Tests of Irreducible Complexity for the Bacterial Flagellum](http://www.discovery.org/scripts/viewDB/index.php?command=view&amp;id=3718&amp;program=CSC%20Responses)."  Evolution News & Views.  April 19, 2006.

_Summary_: All the IDists think that 3/4 of the flagellum proteins are "unique", i.e. do not share homologies with other proteins.  All they are aware of is the homologies to T3SS, which they usually mention while rebutting Kenneth Miller.  Casey Luskin, a late example, cuts the number to 2/3, probably because he is dimly aware that there are some other homologous proteins out there, perhaps because several of us ID skeptics have been mentioning this point repeatedly for several years.

**_The facts_**

What's the truth?  Have a look at [Table 1 of Pallen and Matzke 2006](http://www.nature.com/nrmicro/journal/v4/n10/fig_tab/nrmicro1493_T1.html):


<b>Table 1: Homologies of flagellar proteins</b>

<div class="container-table"><table class="data" border="1" cellpadding="0" cellspacing="0"><colgroup><col align="left"/><col align="left"/><col align="left"/><col align="left"/><col align="left"/><col align="right"/></colgroup><thead valign="top"><tr><th class="last rowsep">Protein</th><th class="last rowsep">Location</th><th class="last rowsep">Function</th><th class="last rowsep">Indispensable?</th><th class="last rowsep">Homologies<sup>*</sup></th><th class="last rowsep">Refs</th></tr></thead><tfoot><tr class="odd"><td colspan="6"><p><sup>*</sup>

Homologies (as evidenced by expected values &lt;1e05) can be confirmed by retrieving the relevant flagellar protein sequences for <span class="i">Escherichia coli</span> K-12 or <span class="i">Salmonella enterica</span> Typhimurium LT2 and carrying out the following:</p></td></tr>

<tr><td colspan="6"><p><sup>||</sup>performing a BLASTp search at the NCBI site under default conditions.</p></td></tr>

<tr><td colspan="6"><p><sup>‡</sup>performing multiple PSI-BLAST iterations at the NCBI site under default conditions;</p></td></tr>

<tr class="odd"><td colspan="6"><p><sup>§</sup>performing multiple iterations of PSI–BLAST at the NCBI site under default conditions, except for adjusting the threshold for inclusion to 0.05 and restricting the taxonomic scope to Enterobacteriaceae, starting with the protein sequences for FlgB, FlgL;</p></td></tr>

<tr class="odd"><td colspan="6" class="last"><p><sup>¶</sup>These similarities fail to achieve unequivocal significance using BLAST/PSI–BLAST under any of the above conditions, but are supported by other structural or functional considerations. T3SS, type III secretion system.</p></td></tr></tfoot>

<tbody valign="top"><tr class="odd"><td>FlgA</td><td>P ring</td><td>Chaperone?</td><td>Absent from Gram-positive bacteria</td><td>CpaB<sup>‡</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a>, <a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B29">29</a></td></tr><tr><td>FlgBCFG</td><td>Rod</td><td>Transmission shaft</td><td>Yes</td><td>FlgBCEFGK<sup>§</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlgD</td><td>Hook</td><td>Hook cap</td><td>Yes</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlgE</td><td>Hook</td><td>Universal joint</td><td>Yes</td><td>FlgBCEFGK<sup>§</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlgH</td><td>L ring</td><td>Bushing</td><td>Absent from Gram-positive bacteria</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlgI</td><td>P ring</td><td>Bushing</td><td>Absent from Gram-positive bacteria</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlgJ</td><td>Rod</td><td>Rod cap; muramidase</td><td>FlgJ N-terminal domain absent from some systems</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlgK</td><td>Hook–filament junction</td><td>Hook-associated protein 1</td><td>Yes</td><td>FlgBCEFGK<sup>§</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlgL</td><td>Hook–filament junction</td><td>Hook-associated protein 3</td><td>Yes</td><td>FliC<sup>§</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlgM</td><td>Cytoplasm and exterior</td><td>Anti-sigma factor</td><td>Absent from <span class="i">Caulobacter</span></td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlgN</td><td>Cytoplasm</td><td>Chaperone</td><td>Undetectable in some systems</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlhA</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>LcrD/YscV<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlhB</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>YscU<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FlhDC</td><td>Cytoplasm</td><td>Transcriptional regulator</td><td>Absent from many systems</td><td>Other activators<sup>‡</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FlhE</td><td>Unknown</td><td>Unknown</td><td>Mutant retains full motility</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliA</td><td>Cytoplasm</td><td>sigma factor</td><td>Absent from <span class="i">Caulobacter</span></td><td>RpoD, RpoH, RpoS<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliB</td><td>Cytoplasm</td><td>N-methylase</td><td>Absent from <span class="i">Escherichia coli</span></td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliC</td><td>Filament</td><td>Flagellin</td><td>Yes</td><td>FlgL<sup>§</sup>, EspA<sup>¶</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a>, <a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B78">78</a></td></tr><tr class="odd"><td>FliD</td><td>Filament</td><td>Filament cap; hook-associated protein 2</td><td>Absent from <span class="i">Caulobacter</span></td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliE</td><td>Rod/basal body</td><td>MS ring–rod junction</td><td>Yes</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliF</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>YscJ<sup>§</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliG</td><td>Peripheral</td><td>Motor</td><td>Yes</td><td>MgtE<sup>¶</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliH</td><td>T3SS apparatus</td><td>Regulates FliI</td><td>Mutant retains some motility</td><td>YscL<sup>*</sup>, AtpFH<sup>¶</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B38">38</a>, <a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B79">79</a></td></tr><tr><td>FliI</td><td>T3SS apparatus</td><td>ATPase for protein export</td><td>Yes</td><td>YscN<sup>||</sup>, AtpD<sup>||</sup>, Rho<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B38">38</a></td></tr><tr class="odd"><td>FliJ</td><td>Cytoplasm</td><td>Chaperone</td><td>Undetectable in some systems</td><td>YscO<sup>¶</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliK</td><td>Hook/basal body</td><td>Controls hook length</td><td>Yes</td><td>YscP<sup>¶</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliL</td><td>Basal body</td><td>Unknown</td><td>Mutant retains full motility</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B80">80</a></td></tr><tr><td>FliM</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>FliN<sup>‡</sup>, YscQ<sup>‡</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliN</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>FliM<sup>‡</sup>, YscQ<sup>‡</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliO</td><td>T3SS apparatus</td><td>Protein export</td><td>Undetectable in some systems</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliP</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>YscR<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliQ</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>YscS<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliR</td><td>T3SS apparatus</td><td>Protein export</td><td>Yes</td><td>YscT<sup>||</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliS</td><td>Cytoplasm</td><td>FliC chaperone</td><td>Absent from <span class="i">Caulobacter</span></td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>FliT</td><td>Cytoplasm</td><td>FliD chaperone</td><td>Absent from many systems</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td>FliZ</td><td>Cytoplasm</td><td>Regulator</td><td>Absent from many systems</td><td>None yet known</td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr class="odd"><td>MotA</td><td>Inner membrane</td><td>Motor</td><td>Yes</td><td>ExbB<sup>‡</sup>, TolQ<sup>‡</sup></td><td><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr><tr><td class="last">MotB</td><td class="last">Inner membrane</td><td class="last">Motor</td><td class="last">Yes</td><td class="last">ExbD<sup>‡</sup>, TolR<sup>‡</sup>, OmpA<sup>‡</sup></td><td class="last"><a href="http://www.nature.com/nrmicro/journal/v4/n10/full/nrmicro1493.html#B25">25</a></td></tr></tbody></table></div>


OK, let's do the math.\*

Total number of proteins listed: **42** 

(this table excludes the chemotaxis proteins; there are ~10 chemotaxis proteins in standard _E. coli_, but the number can range from 0 to 10+ in various bacteria)

Total number thought to be indispensable in modern flagella: **23** (55%)

Total number "unique" (no known homologs): **15** (36%)

Total number of indispensable proteins that are also "unique": **2** (5%)

**_The implications_**

Here are some of the various implications of these findings:

**1.** Scott Minnich, the leading flagellum expert in the ID camp, was severely wrong about the most basic data relevant to the origins of the flagellum, the flagship system of the ID movement.

**2.** Minnich promoted his mistaken view in the video _Unlocking the Mystery of Life_, which has been widely promoted by ID advocates across the country, viewed in countless church basements, IDEA club meetings, etc.

**3.** Stephen Meyer, the Program Director of the Center for ~~the Renewal of~~ Science and Culture at the Discovery Institute, was severely wrong about the most basic data relevant to the origins of the flagellum, the flagship system of the ID movement.

**4.** The paper by Minnich and Meyer repeated this error, and if the paper was "peer-reviewed" at all, it was not by people who knew anything about the most basic data relevant to the origins of the flagellum.

**5.** The error propagated further in Minnich's expert report.

**6.** William Dembski, who put the flagellum on the cover of his book _No Free Lunch_, uses the flagellum in all his talks, and makes it the key example in his only wild attempt to actually "do the calculation" of the probability of the natural origin of a complex biological system, was severely wrong about the most basic data relevant to the origins of the flagellum, the flagship system of the ID movement.

**7.** Discovery Institute spokesman Casey Luskin, although slightly less wrong than everyone else, was severely wrong about the most basic data relevant to the origins of the flagellum, the flagship system of the ID movement, even in an essay where he criticized Kenneth Miller and Judge Jones for getting the flagellum stuff wrong in the _Kitzmiller_ case. 

(And, I personally observed Casey Luskin promulgating this exact mistaken talking point to a gaggle of press at the Kitzmiller trial.  I corrected him publicly right then (I think I said, "Just so you know, everything he said just then is totally wrong"), but to no avail.)

**8.** Casey Luskin's helpers, "Alex Binz, David Klinghoffer, and an un-named biochemist in the University of California system" were also unable to catch the basic error.

**9.** None of the above folks ever bothered to go to the library and check their basic facts.

**10.** Heck, none of these guys bothered to _get on the web_ and check their basic facts.

**11.** No one else in the ID movement, either leaders or fans, ever bothered to check their facts and correct these guys publicly or privately.

**12.** Apparently, everyone in the ID movement just mindlessly copies everyone else's talking points. 

**13.** None of these folks have a friggin' clue what they are blathering about when it comes to flagellum evolution.

That about covers it, I think.  Comments are welcome, as are further examples of this error in the ID literature.

PS: Yes, I am aware that several of the above ID articles contain various Emergency Backup Arguments that say something like "Even if all of the flagellum proteins had homologs, the flagellum still couldn't evolve because of X."  I will address these in more detail later, but all of these arguments are clearly weaker than the "unique parts" argument.  E.g., the most common Emergency Backup Argument is that evolving the necessary regulation is impossible even if the structural proteins are available.  However, Table 1 shows that _none_ of the regulatory proteins are universally conserved across flagella (unless you count FliK -- but FliK has a homolog in nonflagellar T3SS).  

\* Note: the math has been fixed in edit.  That's what I get for posting in the middle of the night.

\*\* Note: This version of the table is slightly different than the published one, which has some blank cells that should read "None yet known."

\[updated the AOP links to permanent links\]
