---
layout: article
title: IBM Discovery Could Shed New Light on Workings of the Human Genome
date: '2006-04-29 19:26:38 -0700'
author: PvM
mt_id: 2159
blog_id: 2
post_id: 2159
basename: ibm_discovery_c
---
IBM researchers have shown how science explores new and innovative approaches to discover DNA patterns which are shared by areas of the human genome that were considered to have little or no influence on its function and areas which do have function.

From the [IBM Press Release](http://domino.research.ibm.com/comm/pr.nsf/pages/news.20060425_dna.html)

> As reported today in the Proceedings of the National Academy of Sciences (PNAS), regions of the human genome that were assumed to largely contain evolutionary leftovers (called "junk DNA") may actually hold significant clues that can add to scientists' understanding of cellular processes. IBM researchers have discovered that these regions contain numerous, short DNA "motifs," or repeating sequence fragments, which also are present in the parts of the genome that give rise to proteins.

The work was also reported in PNAS in an Open Access Article

[Short blocks from the noncoding parts of the human genome have instances within nearly all known genes and relate to biological processes](http://www.pnas.org/cgi/content/abstract/103/17/6605) by isidore Rigoutsos, Tien Huynh, Kevin Miranda, Aristotelis Tsirigos, Alice McHardy, and Daniel Platt, PNAS | April 25, 2006 | vol. 103 | no. 17 | 6605-6610 


> **Abstract**
> 
> Using an unsupervised pattern-discovery method, we processed the human intergenic and intronic regions and catalogued all variable-length patterns with identically conserved copies and multiplicities above what is expected by chance. Among the millions of discovered patterns, we found a subset of 127,998 patterns, termed pyknons, which have additional nonoverlapping instances in the untranslated and protein-coding regions of 30,675 transcripts from 20,059 human genes. The pyknons arrange combinatorially in the untranslated and coding regions of numerous human genes where they form mosaics. Consecutive instances of pyknons in these regions show a strong bias in their relative placement, favoring distances of {approx}22 nucleotides. We also found pyknons to be enriched in a statistically significant manner in genes involved in specific processes, e.g., cell communication, transcription, regulation of transcription, signaling, transport, etc. For {approx}1/3 of the pyknons, the intergenic/intronic instances of their reverse complement lie within 380,084 nonoverlapping regions, typically 60--80 nucleotides long, which are predicted to form double-stranded, energetically stable, hairpin-shaped RNA secondary structures; additionally, the pyknons subsume {approx}40% of the known microRNA sequences, thus suggesting a possible link with posttranscriptional gene silencing and RNA interference. Cross-genome comparisons reveal that many of the pyknons have instances in the 3' UTRs of genes from other vertebrates and invertebrates where they are overrepresented in similar biological processes, as in the human genome. These unexpected findings suggest potential unique functional connections between the coding and noncoding parts of the human genome.

Exciting new findings which show how real science explores 'junk DNA'

The BBC reported on this research in an article titled [Salvage prospect for 'junk' DNA](http://news.bbc.co.uk/2/hi/science/nature/4940654.stm)

> Dr Andrew McCallion, who was not an author on the new paper, commented: "Up until not so long ago, we were under the impression that the vast majority of information in the genome, if not all of it, was encoded in those stretches of DNA that encoded proteins.
> 
> "We now understand there is much more complexity involved," Dr McCallion, from the McKusick-Nathans Institute of Genetic Medicine at the Johns Hopkins University School of Medicine in Baltimore, US, told the BBC News website. 

I am sure ID activists are going to conflate the newly found 'complexity' in the genome with their flawed Design Inference...

But science has already blocked any such design inference by providing a plausible hypothesis

> Gene silencing
> 
> The paper in PNAS suggests that the actual positioning of the motifs is associated with small RNA molecules that are involved with a process called post-transcriptional gene silencing (PTGS).
> 
> "A human embryo starts out as a single fertilised cell and rapidly divides into a widely complex series of cells that become a human being," explained Dr McCallion.
> 
> "Every cell in that human being contains the same complement of genes and what makes each cell different is the precise way that genes are turned on and turned off."
> 
> PTGS turns genes off after the process of transcription has taken place. One way in which this occurs is through "RNA interference", which involves the introduction of double-stranded RNA molecules.
> 
> These trigger the degradation of another type of RNA molecule known as messenger RNA (mRNA), "down-regulating" the gene. During transcription, this molecule encodes and carries information from genes to sites of protein synthesis.
> 
> "These regions may indeed contain structure that we haven't seen before," said Dr Rigoutsos.
> 
> "If indeed one of them corresponds to an active element that is involved in some kind of process, then the extent of cell process regulation that actually takes place is way beyond anything we have seen in the last decade." 

Many websites are reporting on these exciting new findings

> If verified experimentally, the discovery suggests a potential connection between these coding and non-coding parts of the human genome that could have a profound impact on genomic research and provide important insights on the workings of cells. 


[IBM Discovery Could Shed New Light on Workings of the Human Genome](http://www.physorg.com/news65195592.html) at Physorg.com
