---
layout: article
title: KEEP CALM and ASK ABOUT ONIONS
date: '2013-07-01 13:31:16 -0700'
author: Nick Matzke
mt_id: 6772
blog_id: 2
post_id: 6772
basename: keep_calm_and_a
---
<img src="/uploads/2013/junk_DNA_front.png" alt="junk_DNA_front.png" width="100" height="133" style="float: left; margin: 0 20px 20px 0;" class="mt-image-left" />The SMBE meeting is coming up next week in Chicago, and I am going for the first time, in part to present some of my work combining fossils and molecular data in Bayesian dating analyses\*, and partially because I wanted to see the junk DNA symposium in person: [http://smbe2013.org/2013/Scientific-Program/2013-Symposia.aspx](http://smbe2013.org/2013/Scientific-Program/2013-Symposia.aspx)

> **July 11, 2013 (Thursday)**
> Afternoon Symposia (3:00 - 6:30 p.m., coffee break 4:30 - 5:00 p.m.)
> 
> [34. Where Did "Junk" Go? Wojciech Makalowski. Invited Speakers:  Josefa Gonzalez, Valer Gotea, Dixie Mager, Dan Graur, Masafumi Nozawa, Laurence Moran](http://smbe2013.org/2013/Scientific-Program/2013-Symposia.aspx)
> 
> Late Susumu Ohno once said "So much junk DNA in our genome" and the phrase junk DNA was born. For a long time mainstream scientists avoided these parts of the genome. However, over the years the picture slowly started to appear suggesting that the junk DNA hides a genomic treasure. With the completion of the current ENCODE project, junk DNA effectively disappeared because there's no longer useless DNA in the genome. This symposium will discuss the current understanding of these not-so-long-ago obscure areas of the genome, with special attention to transposable elements' activities and their evolutionary consequences. The integral part of the symposium will be general discussion of Ohno's idea and its place in today's biology.

The abstract of this session leaves something to be desired in terms of demonstrating understanding of the real issues here (such as extreme genome size variability).  (Note: the [PDF of talk abstracts](http://smbe2013.org/2013/Scientific-Program/2013-Symposia/Abstracts/SYMPOSIUM-34.aspx) looks better.) But, if some of the participants show some of the same blind spots that the ENCODE project leadership did\*\*, then at least we are in for a good time, since [Dan Graur](http://sandwalk.blogspot.com) and [Larry Moran](http://sandwalk.blogspot.com) will be there to correct them.

I'm hoping there is some science press there, and very interested to see what the reactions of SMBE participants will be.  To stir the pot, and to popularize Ryan Gregory's [Onion Test](http://www.genomicron.evolverzone.com/2007/04/onion-test/) to people who might need it,  I've even [made a t-shirt that some of us will be wearing](http://www.funnyshirts.org/design/b1d3356d02b17d2667dd8098f909a993_40597):

<img src="/uploads/2013/junk_DNA_shirt.png" alt="junk_DNA_shirt.png" width="748" height="389" style="text-align: center; display: block; margin: 0 auto 20px;" class="mt-image-center" />

I've ordered a few extras that I am giving away, [but order your own now!](http://www.funnyshirts.org/design/b1d3356d02b17d2667dd8098f909a993_40597)  If you order rush delivery today, you can still get it by Saturday or Monday, or for a day or two later you could get it shipped to the conference hotel.  (Note: I make no money -- funnyshirts.org has a great "KEEP CALM" template that made this super-easy.)

**Notes**

\* Here's the abstract of the poster I'm giving:

> **Title:** Tighter estimation of hominoid divergence times by Bayesian
> joint analysis of dated fossil morphology and incompletely sorted genes
> Authors: Nicholas J. Matzke1\*†, Joshua G. Schraiber1†, Mark Collard2, Mana Dembo2,
> Melinda A. Yang1†
> 
> **Affiliations:**
> 
> 1 Center for Theoretical and Evolutionary Genomics, Department of Integrative Biology,
> University of California, Berkeley.
> 
> 2 Human Evolutionary Studies Program and Department of Archaeology, Simon Fraser
> University, Burnaby, British Columbia, Canada
> 
> \*Correspondence to: matzke@berkeley.edu
> †Current address: 1005 Valley Life Sciences Building #3140, Berkeley, CA 94720-3140
> 
> **Abstract:** The date of the human-chimp divergence (HCD) remains frustratingly
> uncertain, with estimates collated in TimeTree ranging from 4-10 million years ago
> (Mya), with little evidence of convergence. All hominoid divergence times show similar
> uncertainty, despite their fundamental importance for human genetics and evolution, due
> to difficulties in extrapolating the long-term substitution rate from modern measurements
> of the mutation rate, and uncertainty in the phylogenetic placement of fossil taxa. Here
> we use a hierarchical Bayesian model, implemented in BEAST, to simultaneously
> estimate (1) a dated tree based on discrete morphological characters from 39 fossil and
> living terminal taxa, and (2) a dated species tree of living taxa based on 100 noncoding
> genomic loci evolving under the multispecies coalescent. Only the dates (sometimes
> uncertain) of fossil tips are used to inform the dating of the tree; the problem of the
> subjectivity of prior distributions on node dates is avoided completely. The fossil data
> overwhelm even a strong prior favoring a high substitution rate, despite inclusion of all
> uncertainty in the dating and phylogenetic placement of fossils. The advantage of
> combined analysis is striking: a dated morphology-only analysis yields a 95% highest
> posterior density (HPD) on the human-chimp divergence time of 4.5-8.95 Mya, but in the
> combined analysis the 95% HPD is 4.38-5.54 Mya, despite the uncertain phylogenetic
> position of some fossils. In fossil-rich situations, the precision of estimates of node dates
> may soon approach the timescale of speciation events themselves (less than 1 My).

\*\* I say leadership, since I have heard from many people that many of the participants did not agree with the "80% of the human genome is functional" claim trumpeted to the press from the top.
