---
layout: article
title: 'Accessible research: A tiny bladderwort (that''s a plant with little "bladders")
  genome '
date: '2013-05-14 10:22:20 -0700'
author: M. Wilson Sayres
mt_id: 6742
blog_id: 2
post_id: 6742
basename: accessible_rese_1
---
The genome from a species of bladderwort (_Utricularia gibba_) was [recently published](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12132.html). Ed Yong has a [wonderful summary about the bladderwort genome](http://phenomena.nationalgeographic.com/2013/05/12/flesh-eating-plant-cleaned-junk-minimalist-genome/) paper and its relationship with current debates regarding what is functional ([introduction to the ENCODE project](http://mathbionerd.blogspot.com/2013/04/encode-good-bad-and-ugly.html)). Here's my accessible research introduction:<br />
<br />
The bladderwort is a carnivorous plant with beautiful yellow flowers on top:<br />
<br />



|[<img src="//upload.wikimedia.org/wikipedia/commons/thumb/9/94/Horned_Bladderwort_%284959623968%29.jpg/512px-Horned_Bladderwort_%284959623968%29.jpg" alt="Horned Bladderwort (4959623968)" width="300" />](http://commons.wikimedia.org/wiki/File%3AHorned_Bladderwort_%284959623968%29.jpg)
|This is a captivating "horned bladderwort" (_Utricularia cornuta_), by Jacopo Werther|<br />





<div markdown="block" style="text-align: justify;">

</div>



And curious "bladders" on its roots that it uses to [trap its prey](http://en.wikipedia.org/wiki/Utricularia). <br />

<br />



|[<img src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a4/Common_Bladderwort.jpg/512px-Common_Bladderwort.jpg" alt="Common Bladderwort" width="300" />](http://commons.wikimedia.org/wiki/File%3ACommon_Bladderwort.jpg)
|Bladder traps on the roots of the "common bladderwort" (_Utricularia vulgaris_), by pellaea.




<br />
The genome of this plant is so interesting because it is quite small. The authors go so far as to call it a "minimal" genome.<br />
<br />
**What makes it minimal?**<br />
Often, when talking about genetics, we usually talk about the genes. These are the pieces of DNA that code for proteins. These are "coding" regions. There are also pieces of DNA that do not code for proteins. We call these "non-coding". The genome is the full set of coding and non-coding DNA. Some species have lots of non-coding DNA (like humans, and [onions](http://www.genomicron.evolverzone.com/2007/04/onion-test/%20)). Other species have very little non-coding DNA, including this bladderwort.<br />
<br />
Although the bladderwort has nearly 10,000 more genes than a human, these genes are very compressed and overlapping so that the bladderwort genome and the human genome have about the same amount of coding DNA sites. But, the bladderwort genome has a lot less non-coding DNA, for a total genome size of 87 million bases, while the human genome has bloated amount of non-coding DNA a total genome size of [3,164.7](http://www.ornl.gov/sci/techresources/Human_Genome/project/info.shtml) million bases. That's over 36 times more total DNA than the bladderwort! (Note: the total DNA content does not indicate complexity - whatever that is. Check out the range in genome size of "flowering plants" at the top of [this plot](http://www.mun.ca/biology/scarr/MGA2_02-10.html).) <br />
<br />
**Was the bladderwort genome always so slim?**<br />
No. By comparing with other yummy plant genomes (papaya, grape, tomato and _Arabidopsis_, a flowering mustard weed), and by analyzing the gene content within the bladderwort genome, [Ibarra-Laclette et al.](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12132.html) concluded that the bladderwort genome, like many plants, experienced duplications of its whole genome. Afterward there were some losses of large regions of DNA, but much of the reduction in DNA content from these duplications occured through what they call "microdeletions", where small pieces of DNA here and there were deleted. <br />
<br />
**Are all bladderworts the same?**<br />
Just like there are many species in the taxonomic family Hominidae (orangutans, gorillas, bonobos, chimpanzees and humans), there are many species in the the taxonomic family of bladderworts (called [Lentibulariaceae](http://en.wikipedia.org/wiki/Lentibulariaceae)). Genome shrinking is not unique to this species of bladderwort, but is also not shared across all species of bladderworts. Curiously, genome size in the whole family of bladderworts varies from 60 million base pairs to 1,500 million base pairs. That is a lot of variation considering that genome sizes across all Hominidae are all within the range of 3,000 million base pairs (plants genomes get no respect).<br />
<br />
Although genome size varies quite a bit, the bladderworts look and function in similar ways. So, while some of the noncoding sequence is functional (providing instructions for how and when to turn the genes on and off, in time, in response to the environment, and in particular tissues), it seems highly unlikely that there would be so many more instructions in the bladderwort with 1,500 million base pairs than in the bladderwort with 60 million base pairs. A lot of the non-coding DNA, therefore, is likely also non-functional.<br />
<br />



|[<img src="//upload.wikimedia.org/wikipedia/commons/thumb/8/8e/Utricularia_vulgaris_003.JPG/256px-Utricularia_vulgaris_003.JPG" alt="Utricularia vulgaris 003" width="256" />](http://commons.wikimedia.org/wiki/File%3AUtricularia_vulgaris_003.JPG)
|bladderwort (Utricularia vulgaris) in the water. by H. Zell.




<br />
In the closing line of the bladderwort genome paper:<br />


> In summary,&nbsp;_U. gibba_&nbsp;genome architecture demonstrates that angiosperms can evolve diverse gene landscapes while overall genome size contracts, not only during expansions. Furthermore, in contrast to recent publications that highlight a crucial functional role of non-coding DNA in complex organisms such as animals<sup>[24](http://www.nature.com/nature/journal/vaop/ncurrent/full/nature12132.html#ref24)</sup>, the necessary genomic context required to make a flowering plant may not require substantial hidden regulators in the non-coding 'dark matter' of the genome.


<br />
<br />
<br />
Enrique Ibarra-Laclette, Eric Lyons, Gustavo Hernández-Guzmán, Claudia Anahí Pérez-Torres, Lorenzo Carretero-Paulet, Tien-Hao Chang, Tianying Lan, Andreanna J. Welch, María Jazmín Abraham Juárez, June Simpson, Araceli Fernández-Cortés, Mario Arteaga-Vázquez, Elsa Góngora-Castillo, Gustavo Acevedo-Hernández, Stephan C. Schuster, Heinz Himmelbauer, André E. Minoche, Sen Xu, Michael Lynch, Araceli Oropeza-Aburto, Sergio Alan Cervantes-Pérez, María de Jesús Ortega-Estrada, Jacob Israel Cervantes-Luevano, Todd P. Michael, Todd Mockler, Douglas Bryant, Alfredo Herrera-Estrella, Victor A. Albert,&nbsp; & Luis Herrera-Estrella. **&nbsp;**<br />
**Architecture and evolution of a minute plant genome**. 2013. Nature<br />
&nbsp;[http://dx.doi.org/10.1038/nature12132](http://dx.doi.org/10.1038/nature12132)
