---
layout: article
title: Scurvy, Guinea Pigs, and You
date: '2004-09-05 03:36:09 -0700'
author: Reed A. Cartwright
mt_id: 427
blog_id: 2
post_id: 427
basename: scurvy_guinea_p
---
In my last post, [Common Design Errors](http://pandasthumb.org/archives/2004/08/common-design-e.html), I proposed a problem for biblical creation.  I received one response from a creationist, who cited Inai et al. (2003).  This paper compared the largest set of homologous exons between humans, guinea pigs, and rats.  You see, guinea pigs, like most primates and a few other taxa, lack L-guluno-gamma-lactone oxidase.  Two sections were quoted to me.

> When the human and guinea pig sequences (647 nucleotides in total) of the regions of exons 4, 7, 9, 10, and 12 were compared, we found 129 and 96 substitutions in humans and guinea pigs, respectively, when compared with the rat sequences (Fig. 2).  The same substitutions from rats to both human and guinea pigs occurred at 47 nucleotide positions among the 129 positions where substitutions occurred in the human sequences.  A high percentage of the same substitutions in the total substitutions (36%) indicates that there were many hot spots for nucleotide substitution throughout the sequences examined.
> 
> p. 316

> Assuming an equal chance of substitution throughout the sequences, the probability of the same substitutions in both humans and guinea pigs occurring at the observed number of positions and more was calculated to be 1.84 x 10-12.  This extremely small probability indicates the presence of many mutational hot spots in the sequences.
> 
> p. 317

This was to support the following response by the reader

> Obviously if the same substitutions are occurring independently in the primate and guinea pig lineages, they could have arisen independently within the primates as well.

which was followed up by

> If single sustitutions can occur independently, how do we know that larger deletions couldn't as well?

However, the sections quoted from Inai et al. (2003) suffer from a major methodological error; they failed to consider that substitutions could have occured in the rat lineage after the splits from the other two.  The researchers actually clustered substitutions that are specific to the rat lineage with separate substitutions shared by guinea pigs and humans.  To illustrate this point, I am going to use sequences homologous to rat exon 10, for which we have the most data from other species.

The following figures list a section of rat exon 10 and the sections from ten species homologous to it.  The sequences were retreived from NCBI and aligned using ClustalX.  In the figure, periods represent nucleotides that are the same as  rat nucleotides.

```                                                                                                                  1
                     1         2          3         4          5         6          7         8          9         0          1         2          3         4          5         6     
            12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345
Rat         GGAGAAGACCAAGGAGGCCC TACTGGAGCTAAAGGCCATG CTGGAGGCCCACCCCAAAGT GGTAGCCCACTACCCCGTAG AGGTGCGCTTCACCCGAGGC GATGACATTCTGCTGAGCCC CTGCTTCCAGAGGGACAGCT GCTACATGAACATCATTATG TACAG
Guinea Pig  A................... .G........G...AG.... .....A..T........G.. ..C............T..G. G...............G..G ..C.....C........... ..C................. ..............TGC..A .....
Human       AA.........C........ .G........G......G.. ..............TG.G.. ...GT.........TG..G. G...A.........T.GA.G -.......C..A........ ..........T........C .....C.........ACC.. .....```

If I performed the same analysis as Inai et al. (2003), I would conclude that there are ten positions where humans and guinea pigs experienced separate substitutions of the same nucleotide, otherwise known as shared, derived traits.  These positions are 1, 22, 31, 58, 79, 81, 97, 100, 109, 157.  However, most of these are shown to be substitutions in the rat lineage when we look at larger samples of species.

```                                                                                                                  1
                     1         2          3         4          5         6          7         8          9         0          1         2          3         4          5         6     
            12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345678901234567890 12345
Rat         GGAGAAGACCAAGGAGGCCC TACTGGAGCTAAAGGCCATG CTGGAGGCCCACCCCAAAGT GGTAGCCCACTACCCCGTAG AGGTGCGCTTCACCCGAGGC GATGACATTCTGCTGAGCCC CTGCTTCCAGAGGGACAGCT GCTACATGAACATCATTATG TACAG
Mouse       .................... .G.................. .................G.. ..................G. ...................T ........C........... G................... .................... .....
Guinea Pig  A................... .G........G...AG.... .....A..T........G.. ..C............T..G. G...............G..G ..C.....C........... ..C................. ..............TGC..A .....
Human       AA.........C........ .G........G......G.. ..............TG.G.. ...GT.........TG..G. G...A.........T.GA.G -.......C..A........ ..........T........C .....C.........ACC.. .....
Chimp       AA.........C........ .G........G......... ...............G.G.. ...GT.........TG..G. G.C.A.........T.GA.G -.......C..A........ ..........C........C .....C.........ACC.. .....
Oragutan    AA.........C........ .G........G......... ..............TG.G.. ...GT..........G..G. G..............AGA.G -.....G.C..A........ ..........CA.......C ....TC.........ACC.. .....
Macaque     AA.........CA.G..... .G......A.G......... ..............TG.G.. ...GT.......A..G..G. G..............A...G -.......CA.A........ ..........CA........ .....C..G......ACC.. .....
Cow         A...........A....... .G........G......... ........GAG......G.. A..G..............G. ....A........T..C..G ..C.....C........... ..........C.A....... ...........G....C... .....
Pig         A................... .C........G......... .................G.. ...G..............G. .............T..G.CG ..C.....C........... .................... ................C... .....
Chicken     T........A.....A..A. .G........G.....TGCC ......AA.A.......GA. ...G...........T..G. ..........TG.T....CG .....G..CTG......... .................... ................C... .....
Tiger Shark TA..C....T.GA.CA..T. .GGA.C....G....ATTG. .....CAA.A.T..T..T.. .CG...A..T.TT..T..C. ....T..G..TGTT..T.CA ..C..T........C..... ...T.A.AGACA........ .......T........C... .....```

When we look at this larger data table, only one position of the ten, 81, stands out as a possible case of a shared derived trait, one position, 97, is inconclusive, and the other eight positions are more than likely shared ancestral sites.   With this additional phylogenetic information, I have shown that the "hot spots" Inai et al. (2003) found are not well supported.  Therefore, the explaination given to me by the creationist who responded does not work.


* Inai Y et al. (2003), "The Whole Structure of the Human Non-Functional L-Guluno-gamma-Lactone Oxidase Gene - the Gene Responsible for Scurvy - and the Evolution of Repetitive Sequences Thereon," J Nutr Sci Vitaminology
49:315-319.
