---
layout: article
title: 'Evolvability: ALU repeats and hominid phylogenetics'
date: '2004-10-13 18:49:26 -0700'
author: PvM
mt_id: 490
blog_id: 2
post_id: 490
basename: evolvability_al
---
While researching the hot topic of ALU repeats for my posting on evolvability, I ran across the following paper

[Alu elements and hominid phylogenetics](http://batzerlab.lsu.edu/Publications/Salem%20and%20Ray%20et%20al%202003%20-%20Alus%20and%20hominid%20phylogeny.pdf) by Abdel-Halim Salem, David A. Ray, Jinchuan Xing, Pauline A. Callinan, Jeremy S. Myers, Dale J. Hedges, Randall K. Garber, David J. Witherspoon, Lynn B. Jorde, and Mark A. Batzer published in PNAS October 28, 2003 vol. 100 no. 22 12787-12791 

Let me add a disclaimer that I am a novice when it comes to evolutionary biology. While I am familiar with the term such as ALU repeats, SINE and retroposons, I am not by any means an expert. Nevertheless, I like to share my research and learning with the readers of Panda's Thumb, in the hope that 1) people like me who are similarly interested in learning more about "what is hot" in evolutionary theory can learn about some of the details 2) others, more capable than me, can add their comments, suggestions and objections.

ALU repeats are transponson related repeats about 250-300 base pairs long and unique to the genome of primates. ALU repeats are very common, perhaps as many as one ALU repeat per 3000 base pairs. In the human genome more than half a million ALU repeats exist. ALU repeats have been implicated in various diseases. Research suggests that ALU repeats may be responsible for up to 0.1% of the human genetic disorders (See [Alu repeats and human disease](http://batzerlab.lsu.edu/Publications/Deininger%20and%20Batzer%201999%20Mol%20Genetics%20and%20Metabolism.pdf) Deininger PL, Batzer MA.Mol Genet Metab. 1999 Jul;67(3):183-93)). And although initially thought to be 'junk DNA' evidence suggests that ALU repeats have [a function](http://www.deccanherald.com/deccanherald/jan222004/snt2.asp). 

In the paper, the researchers describe how they use ALU repeates in hominids to infer the phylogenetic tree. ALU repeats are believed to be very suitable for tree reconstruction since they are essentially unidirectional, in other words, once they appear, they are hard to remove from the genome.

Earlier research showed that the phylogeny for hominids was not without controversy

> Resolving the relationships among human (H), chimpanzee (C), and gorilla (G) (i.e., the trichotomy problem) has been particularly difficult. Which of the four possible relationships, ((H,C)G), ((H,G)C), ((C,G)H), and (H,C,G), reflects the true phylogeny of the three species? The consensus approach identifies the chimpanzee as the nearest living relative of humans, but the evidence supporting this conclusion is neither universal nor overwhelming (22-26). 
> 
> 
> [Alu elements and hominid phylogenetics](http://batzerlab.lsu.edu/Publications/Salem%20and%20Ray%20et%20al%202003%20-%20Alus%20and%20hominid%20phylogeny.pdf) Abdel-Halim Salem et al

Using ALU repeats, the researchers show that:

> Here, we present the first application of SINEs to fully elucidate the phylogeny of the hominid lineage and present the strongest evidence to date for phylogenetic relationships among the hominid lineages. Of the 133 Alu insertion loci, 95 were unambiguously informative for determining the relative divergence of each of the major lineages. In addition, seven Alu insertions informative with regard to the Homo-Pan-Gorilla trichotomy unambiguously support monophyly of humans and chimpanzees. Finally, six loci discovered by using searches of the currently available chimpanzee genomic sequences unambiguously join the two Pan species to the exclusion of Homo. Only a single insertion from the entire data set showed any potential evidence of insertion homoplasy; however, the distribution of this element appears to be the result of lineage sorting of an ancestral polymorphism.
> 
> 
> [Alu elements and hominid phylogenetics](http://batzerlab.lsu.edu/Publications/Salem%20and%20Ray%20et%20al%202003%20-%20Alus%20and%20hominid%20phylogeny.pdf) Abdel-Halim Salem et al

The following [website](http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/T/Transposons.html) discusses an interesting hypothesis (I will discuss ALU repeats, and transposons in more detail in my discussion of the evolution of evolvability). For the moment, consider this a teaser.

> Retrotransposons often carry some additional sequences at their 3' end as they insert into a new location. Perhaps these occasionally create new combinations of exons, promoters, and enhancers that benefit the host.
> 
> Example:
> 
> 
> 
> * Thousands of our Alu elements occur in the introns of structural genes.
> * Some of these contain sequences that when transcribed into the primary transcript are recognized by the spliceosome.
> * These can then be spliced into the mature mRNA creating a
> * new exon, which will be transcribed into a new protein product.
> * Alternative splicing can provide not only the new mRNA (and thus protein) but also the old.
> * In this way, nature can try out new proteins without the risk of abandoning the tried-and-true old one.
