---
layout: article
title: 'Evolvability: RNA and The Gems of "Junk" DNA'
date: '2004-10-27 23:25:09 -0700'
author: PvM
mt_id: 506
blog_id: 2
post_id: 506
basename: evolvability_rn
---
Historically, the central dogma in molecular biology has been that the genetic information in DNA is transcribed into intermediate RNA which are translated into amino acids to form proteins.  Proteins were seen as the primary regulators of the expression of genes. While this picture appears to be correct for prokaryotes, a different picture arises for eukaryotes.

Mathematical considerations have shown that while generating complexity is simple, controlling it isn't. The amount of regulation needed tends to scale as the quadratic of the number of genes. The genome size of prokaryotes seems to be [limited by these considerations](http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=sef.figgrp.544). 

<img src="http://www.ncbi.nlm.nih.gov/books/bookres.fcgi/sef/ch9f1.gif" alt="" />

How to resolve the inherent limitation to complexity found in prokaryotes?  While genome size for prokaryotes is limited by the need for regulation of complexity, eukaryotes seem to have found a way to increase complexity beyond the limitations of the genome. Eukaryotes have an additional, parallel, system where **RNA** regulates DNA, RNA, and proteins.

> Eukaryotes must have found a solution to this problem. Logic and the available evidence suggest that the rise of multicellular organisms over the past billion years was a consequence of the transition to a new control architecture based largely on endogenous digital RNA signals. It would certainly help explain the phenomenon of the Cambrian explosion about 525 million years ago, when invertebrate animals of jaw-dropping diversity evolved, seemingly abruptly, from much simpler life. Indeed, these results suggest a general rule with relevance beyond biology: organized complexity is a function of regulatory information--and, in virtually all systems, as observed by Marie E. Csete, now at Emory University School of Medicine, and John C. Doyle of the California Institute of Technology, explosions in complexity occur as a result of advanced controls and embedded networking.
> 
> 
> [Sequence-Evolution-Function: Computational Approaches in Comparative Genomics](http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=sef.figgrp.544)

John Mattick reports on the 'hidden role of RNA and Junk DNA<sup>\[1\]</sup>' in the October 2004 issue of Scientific American<sup>\[2\]</sup>.

Mattick wrote:

> A growing library of results reveals that the central dogma is woefully incomplete for describing the molecular biology of eukaryotes. Proteins do play a role in the regulation of eukaryotic gene expression, yet a hidden, parallel regulatory system consisting of RNA that acts directly on DNA, RNAs and proteins is also at work. This overlooked RNA-signaling network may be what allows humans, for example, to achieve structural complexity far beyond anything seen in the unicellular world.

These ideas are of course not without controversy but they would help explain many paradoxes. Such as the C-Paradox, the observation that the complexity of an organism does not scale with the number of protein coding genes. 

In my explorations of the concept of evolvability, I have shown that evolvability depends not only on the neutrality of the genotype-phenotype mapping but also on the concept of 'neighborhood'.  Transposons change the traditional character of "neighborhood" of point mutations. This has an important impact on evolvability. While evolution may have ended up in a local maximum under point mutations, transposons can 'jump' to higher fitness regions.

The problem with transposons and Junk DNA is that they have remained almost 'invisible'. New techniques have shown us evidence of many exciting new areas, helping us resolve many of the paradoxes in biology. 

And the relevance to intelligent design? Since ID relies strongly on our ignorance, these paradoxes inevitably have led ID proponents to conclude that 'evolutionary theory' cannot explain these observations and 'thus intelligent design' was invoked. We see that with increased understanding, the gaps in which ID can 'fit' are becoming narrower and narrower.  

On the other hand we see how the Neo-Darwinian synthesis had limited the variability in the genome to coding DNA. Unaware of the many of the hidden gems. But science moves on and eventually found the hard to detect evidence that may help explain the amount of "junk DNA". These new findings, rather than limiting evolutionary theory have expanded it, showing the main difference between science and intelligent design when it comes to new knowledge.

*********


# Useful resources


* **Junk DNA**: "Junk DNA" refers to non-coding DNA, found mostly in eukaryotes. Recent data is showing  increasing evidence that some "junk DNA" [may have a function](http://www.mindfully.org/GE/2003/Junk-GenomeNov03.htm) after all.  
First of all, it is important to understand the concept of Junk DNA. [Junk DNA](http://www.wordiq.com/definition/Junk_DNA) refers to DNA sequences which are not translated into proteins.
Of course this does not mean that all junk DNA will have a function, it seems reasonable that pseudogenes for instance will [remain historical artifacts](http://www.talkorigins.org/faqs/molgen/).
* [The hidden genetic program of compex organisms](http://faculty.dbcc.edu/tulsian/interestingReading/biology/The-hidden-genetic-program-of-complex-organisms-qpdf.pdf) John S. Mattick, Scientific American, October 2004 p 60-67
* [Challenging the Dogma: The Hidden Layer of Non-Protein-Coding RNAs In Complex Organisms](http://www.binf.ku.dk/users/tlitman/binf/binf2kursus/Bioessays_2003_v25_p930.pdf) John S. Mattick in BioEssays, Vol. 25, No. 10, pages 930-939; October 2003.
* [The Unseen Genome: Gems among the Junk.](http://www.phy.ntnu.edu.tw/~cchen/class/scitran/genome.pdf) W. Wayt Gibbs in Scientific American, Vol. 289, No. 5, pages 46-53; November 2003.
* Nowak R. **Mining treasures from 'junk DNA'**. Science. 1994 Feb 4;263(5147):608-10.
> The protein-coding portions of the genes account for only about 3% of the DNA in the human genome; the other 97% encodes no proteins. Most of this enormous, silent genetic majority has long been thought to have no real function - hence its name: 'junk DNA.; But one researcher's trash is another researcher's treasure..."
* [Non-genic evolution and selection in the human genome or: "Junk DNA"](http://www.stats.ox.ac.uk/~hein/HumanGenome/hg.pdf) Gerton Lunter. Good overview
*     Genomic scrap yard: how genomes utilize all that junk. [Makalowski W.](http://www.bio.psu.edu/directory/homepages/wxm23.action) Gene. 2000 Dec 23;259(1-2):61-7. 
> Interspersed repetitive sequences are major components of eukaryotic genomes. Repetitive elements comprise over 50% of the mammalian genome. Because the specific function of these elements remains to be defined and because of their unusual 'behaviour' in the genome, they are often quoted as a selfish or junk DNA. Our view of the entire phenomenon of repetitive elements has to now be revised in the light of data on their biology and evolution, especially in the light of what we know about the retroposons. I would like to argue that even if we cannot define the specific function of these elements, we still can show that they are not useless pieces of the genomes. The repetitive elements interact with the surrounding sequences and nearby genes. They may serve as recombination hot spots or acquire specific cellular functions such as RNA transcription control or even become part of protein coding regions. Finally, they provide very efficient mechanism for genomic shuffling. As such, repetitive elements should be called genomic scrap yard rather than junk DNA. Tables listing examples of recruited (exapted) transposable elements are available at [this link](http://www.ncbi.nlm.gov/Makalowski/ScrapY)
